<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><title>Differential abundance — differentialAbundance • MicrobiomeDB</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Differential abundance — differentialAbundance"><meta property="og:description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Differential abundance</h1>
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/R/method-differentialAbundance.R" class="external-link"><code>R/method-differentialAbundance.R</code></a></small>
      <div class="d-none name"><code>differentialAbundance-methods.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (<code><a href="Maaslin2.html">MicrobiomeDB::Maaslin2</a></code>) and DESeq2 (<code>DESeqDataSetFromCollection</code>).</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,missingOrNULL,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,function</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,character</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>data</dt>
<dd><p>AbundanceData object</p></dd>


<dt>covariate</dt>
<dd><p>character vector giving the name of a metadata variable of interest. If this
variable has only two values, you do not need to provide functions for arguments <code>groupA</code> and <code>groupB</code>.</p></dd>


<dt>groupA</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupA. In the specific case of <code>covariate</code> being a categorical variable,
this can be a character vector of values to be included in groupA.</p></dd>


<dt>groupB</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupB. If not provided, groupB will be the complement of groupA. In the specific
case of <code>covariate</code> being a categorical variable, this can be a character vector of values to be included in groupB.</p></dd>


<dt>method</dt>
<dd><p>string defining the the differential abundance method. Accepted values are 'DESeq2' and 'Maaslin2'.
Default is 'Maaslin2', as 'DESeq2' only supports counts.</p></dd>


<dt>verbose</dt>
<dd><p>boolean indicating if timed logging is desired</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>ComputeResult object</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co">## a continuous variable</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>         <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.354675 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.432035 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.433929 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.507152 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.510971 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.511534 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.512922 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.525684 INFO::Formula for fixed effects: expr ~  breastfed_duration</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.526456 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.526944 INFO::Total samples in data: 2533</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.527426 INFO::Min samples required with min abundance for a feature not to be filtered: 253.300000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.543246 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.544052 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Candidatus.Arthromitus, Candidatus.Obscuribacter, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Faecalicoccus, Faecalitalea, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavobacterium, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Marvinbryantia, Massilia, Megamonas, Merdibacter, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Myxococcus, NK4A214.group, Nakamurella, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phaseolibacter, Phocea, Phyllobacterium, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudoflavonifractor, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tetragenococcus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.550476 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.551 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.551491 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.599107 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.60192 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.669098 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.669845 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.674149 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.678066 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.681969 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.685966 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.689877 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.693765 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.697645 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.701549 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.705445 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.70932 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.713206 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.71709 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.721004 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.724887 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.728789 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.732666 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.736588 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.740482 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.744374 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.748251 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.752161 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.756047 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.759943 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.763841 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.767718 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.771616 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.775522 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.779428 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.783319 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.787225 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.791131 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.795037 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.79892 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.802835 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.806702 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.81058 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.814483 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.818376 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.822272 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.826171 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.830045 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.833933 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.837814 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.841778 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.845693 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.849593 INFO::Fitting model to feature number 47, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.853507 INFO::Fitting model to feature number 48, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.857406 INFO::Fitting model to feature number 49, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.861284 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.865146 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.869023 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.87296 INFO::Fitting model to feature number 53, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.876881 INFO::Fitting model to feature number 54, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.880774 INFO::Fitting model to feature number 55, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.884636 INFO::Fitting model to feature number 56, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.888549 INFO::Fitting model to feature number 57, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.892453 INFO::Fitting model to feature number 58, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.896338 INFO::Fitting model to feature number 59, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.90021 INFO::Fitting model to feature number 60, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.904088 INFO::Fitting model to feature number 61, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.907988 INFO::Fitting model to feature number 62, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.911882 INFO::Fitting model to feature number 63, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.915771 INFO::Fitting model to feature number 64, Phascolarctobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.919647 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.923537 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.927411 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.931304 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.935222 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.939105 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.942961 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.946851 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.950712 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.954618 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.95866 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.962559 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.966425 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.97034 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.97423 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.978131 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.982013 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.985911 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.989835 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.993719 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:38.997607 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.001527 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.005427 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.009368 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.013261 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.017174 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.021066 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.024976 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.028851 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.032767 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.036672 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.040601 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.044495 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.048423 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.052319 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.056214 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.060125 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.064066 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.068291 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.08521 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.089154 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.093091 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.097041 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.114129 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.121735 INFO::Writing filtered data to file /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.301572 INFO::Writing filtered, normalized data to file /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.481041 INFO::Writing filtered, normalized, transformed data to file /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.767457 INFO::Writing residuals to file /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.821706 INFO::Writing fitted values to file /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.840801 INFO::Writing all results to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.842456 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fa2ab5bc99/significant_results.tsv</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.844283 Completed method=Maaslin2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.847425 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  Maaslin2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## same variable, using DESeq2</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>             <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>             <span class="st">'16S (V4) Genus (Absolute taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'DESeq2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:39.89592 Integer values detected. Converting collection to AbsoluteAbundanceData</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:40.037618 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:40.168538 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:26:40.170501 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> converting counts to integer mode</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>some variables in design formula are characters, converting to factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> using pre-existing size factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> gene-wise dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> mean-dispersion relationship</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> final dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- replacing outliers and refitting for 267 genes</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- DESeq argument 'minReplicatesForReplace' = 7 </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- original counts are preserved in counts(dds)</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.456458 Completed method=DESeq2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.459526 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  DESeq2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 3 values, one of which we exclude</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Russia"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Finland"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.758107 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.761795 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.762301 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.763765 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.776998 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.777859 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.778354 INFO::Total samples in data: 2692</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.778818 INFO::Min samples required with min abundance for a feature not to be filtered: 269.200000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.79522 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.796322 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asticcacaulis, Aureimonas, Bacillus, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Marinilactibacillus, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylobacillus, Methylobacterium.Methylorubrum, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, Paenibacillus, Paeniclostridium, Paludicola, Papillibacter, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Piscinibacter, Planifilum, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinivibrio, Sulfurospirillum, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gastranaerophilales, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.803022 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.803611 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.804072 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.854946 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.857981 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.91962 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.920441 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.92493 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.929071 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.933132 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.937186 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.957489 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.961711 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.965838 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.9699 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.974006 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.97809 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.982177 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.986365 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.99133 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.995579 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:19.99973 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.00385 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.008005 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.012086 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.016184 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.020267 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.024337 INFO::Fitting model to feature number 22, Corynebacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.028382 INFO::Fitting model to feature number 23, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.032423 INFO::Fitting model to feature number 24, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.036491 INFO::Fitting model to feature number 25, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.040546 INFO::Fitting model to feature number 26, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.044631 INFO::Fitting model to feature number 27, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.048726 INFO::Fitting model to feature number 28, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.052853 INFO::Fitting model to feature number 29, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.057012 INFO::Fitting model to feature number 30, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.06113 INFO::Fitting model to feature number 31, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.065235 INFO::Fitting model to feature number 32, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.069481 INFO::Fitting model to feature number 33, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.073705 INFO::Fitting model to feature number 34, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.077833 INFO::Fitting model to feature number 35, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.08198 INFO::Fitting model to feature number 36, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.086082 INFO::Fitting model to feature number 37, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.090215 INFO::Fitting model to feature number 38, Gordonibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.094288 INFO::Fitting model to feature number 39, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.09838 INFO::Fitting model to feature number 40, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.102541 INFO::Fitting model to feature number 41, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.106588 INFO::Fitting model to feature number 42, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.110659 INFO::Fitting model to feature number 43, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.114756 INFO::Fitting model to feature number 44, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.118834 INFO::Fitting model to feature number 45, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.122924 INFO::Fitting model to feature number 46, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.127246 INFO::Fitting model to feature number 47, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.131416 INFO::Fitting model to feature number 48, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.135552 INFO::Fitting model to feature number 49, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.139651 INFO::Fitting model to feature number 50, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.143866 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.147933 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.152024 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.156124 INFO::Fitting model to feature number 54, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.160222 INFO::Fitting model to feature number 55, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.164372 INFO::Fitting model to feature number 56, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.168474 INFO::Fitting model to feature number 57, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.17255 INFO::Fitting model to feature number 58, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.176651 INFO::Fitting model to feature number 59, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.180746 INFO::Fitting model to feature number 60, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.184809 INFO::Fitting model to feature number 61, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.18884 INFO::Fitting model to feature number 62, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.192872 INFO::Fitting model to feature number 63, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.196908 INFO::Fitting model to feature number 64, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.200928 INFO::Fitting model to feature number 65, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.204949 INFO::Fitting model to feature number 66, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.208986 INFO::Fitting model to feature number 67, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.213038 INFO::Fitting model to feature number 68, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.217081 INFO::Fitting model to feature number 69, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.221178 INFO::Fitting model to feature number 70, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.225246 INFO::Fitting model to feature number 71, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.229317 INFO::Fitting model to feature number 72, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.233362 INFO::Fitting model to feature number 73, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.237422 INFO::Fitting model to feature number 74, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.241455 INFO::Fitting model to feature number 75, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.24551 INFO::Fitting model to feature number 76, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.249542 INFO::Fitting model to feature number 77, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.253596 INFO::Fitting model to feature number 78, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.257652 INFO::Fitting model to feature number 79, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.261719 INFO::Fitting model to feature number 80, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.265716 INFO::Fitting model to feature number 81, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.269762 INFO::Fitting model to feature number 82, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.27383 INFO::Fitting model to feature number 83, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.277858 INFO::Fitting model to feature number 84, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.281882 INFO::Fitting model to feature number 85, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.285886 INFO::Fitting model to feature number 86, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.289904 INFO::Fitting model to feature number 87, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.293928 INFO::Fitting model to feature number 88, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.297928 INFO::Fitting model to feature number 89, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.302009 INFO::Fitting model to feature number 90, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.306011 INFO::Fitting model to feature number 91, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.310003 INFO::Fitting model to feature number 92, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.313998 INFO::Fitting model to feature number 93, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.31801 INFO::Fitting model to feature number 94, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.322016 INFO::Fitting model to feature number 95, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.32606 INFO::Fitting model to feature number 96, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.330086 INFO::Fitting model to feature number 97, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.334111 INFO::Fitting model to feature number 98, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.338133 INFO::Fitting model to feature number 99, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.342193 INFO::Fitting model to feature number 100, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.346197 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.350263 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.354254 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.35827 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.362267 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.366292 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.370273 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.3858 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.391052 INFO::Writing filtered data to file /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.591493 INFO::Writing filtered, normalized data to file /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:20.799631 INFO::Writing filtered, normalized, transformed data to file /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.133443 INFO::Writing residuals to file /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.193472 INFO::Writing fitted values to file /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.213447 INFO::Writing all results to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.215459 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fa7880bf03/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## this case of categorical variables also has a 'short cut'</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="st">"Russia"</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Finland"</span>,<span class="st">"Estonia"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.557191 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.561003 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.561535 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.563294 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.578918 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.579779 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.580273 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.580736 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.601202 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.602337 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.609939 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.610486 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.610937 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.673239 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.676303 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.750995 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.751872 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.756959 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.761769 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.76644 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.771098 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.775769 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.780365 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.784936 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.789487 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.794043 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.798553 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.803141 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.807693 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.812193 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.816679 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.82122 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.825713 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.830222 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.834734 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.839314 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.84389 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.848342 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.852814 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.857286 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.861784 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.866236 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.870758 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.875304 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.879812 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.884286 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.888797 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.893297 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.897786 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.902261 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.906736 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.911263 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.915782 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.937187 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.94199 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.946635 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.9513 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.955889 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.960565 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.96511 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.969673 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.974206 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.97876 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.983352 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.987858 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.992417 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:21.996938 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.001508 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.006022 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.010529 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.01509 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.019606 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.024148 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.028939 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.033553 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.038136 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.042754 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.047606 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.052194 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.056853 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.061485 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.066085 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.070618 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.075172 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.079667 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.084224 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.088735 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.093219 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.097695 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.102184 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.106672 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.111181 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.115729 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.120245 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.124762 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.129233 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.133754 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.138247 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.142754 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.147293 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.151893 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.1565 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.162835 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.170959 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.175909 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.180538 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.185094 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.189592 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.194092 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.198569 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.203087 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.207582 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.212154 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.216707 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.221382 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.225924 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.230433 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.234949 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.239489 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.243988 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.248458 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.252946 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.257444 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.273529 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.279005 INFO::Writing filtered data to file /tmp/Rtmp0cUwdW/maaslin17fa698189a4/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.500044 INFO::Writing filtered, normalized data to file /tmp/Rtmp0cUwdW/maaslin17fa698189a4/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:22.72721 INFO::Writing filtered, normalized, transformed data to file /tmp/Rtmp0cUwdW/maaslin17fa698189a4/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.095223 INFO::Writing residuals to file /tmp/Rtmp0cUwdW/maaslin17fa698189a4/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.165176 INFO::Writing fitted values to file /tmp/Rtmp0cUwdW/maaslin17fa698189a4/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.188885 INFO::Writing all results to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fa698189a4/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.190703 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fa698189a4/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 2 values</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       <span class="st">"delivery_mode"</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span> </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.567304 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.571088 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.571625 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.573413 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.589094 INFO::Formula for fixed effects: expr ~  delivery_mode</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.590049 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.590571 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.591089 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.611936 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.613071 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.620848 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.621438 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.62189 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.683393 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.686413 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.760365 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.76111 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.766123 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.770842 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.775558 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.78016 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.784749 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.789337 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.793937 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.798473 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.803041 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.807606 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.812107 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.816597 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.821106 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.825587 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.830082 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.834546 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.839114 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.843683 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.848154 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.852676 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.857162 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.861626 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.866081 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.870608 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.875063 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.879487 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.883981 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.888432 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.892877 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.897359 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.901837 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.906269 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.910758 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.91524 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.919688 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.924127 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.928576 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.933032 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.937474 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.941951 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.946419 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.950843 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.955307 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.959842 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.964273 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.968708 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.973156 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.977586 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.982012 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.986427 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.990868 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.995291 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:23.999726 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.004152 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.008571 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.013007 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.017446 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.021862 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.026298 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.03071 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.035189 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.039586 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.044056 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.048477 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.052933 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.057381 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.061829 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.066254 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.071293 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.076018 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.080502 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.084976 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.089471 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.093902 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.098333 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.102801 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.107286 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.111691 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.116114 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.120548 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.124973 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.129382 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.133823 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.138201 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.142655 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.147121 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.151543 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.155948 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.160385 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.164858 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.169287 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.173702 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.178092 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.182528 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.186978 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.191406 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.195819 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.200232 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.204632 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.209005 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.213434 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.217815 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.241028 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.245787 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.250403 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.254933 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.271209 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.276549 INFO::Writing filtered data to file /tmp/Rtmp0cUwdW/maaslin17fad5eb847/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.496865 INFO::Writing filtered, normalized data to file /tmp/Rtmp0cUwdW/maaslin17fad5eb847/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:24.721454 INFO::Writing filtered, normalized, transformed data to file /tmp/Rtmp0cUwdW/maaslin17fad5eb847/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:25.084082 INFO::Writing residuals to file /tmp/Rtmp0cUwdW/maaslin17fad5eb847/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:25.155418 INFO::Writing fitted values to file /tmp/Rtmp0cUwdW/maaslin17fad5eb847/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:25.179577 INFO::Writing all results to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fad5eb847/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 18:27:25.181415 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/Rtmp0cUwdW/maaslin17fad5eb847/significant_results.tsv</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

