<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><title>Differential abundance — differentialAbundance • MicrobiomeDB</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Differential abundance — differentialAbundance"><meta property="og:description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Differential abundance</h1>
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/R/method-differentialAbundance.R" class="external-link"><code>R/method-differentialAbundance.R</code></a></small>
      <div class="d-none name"><code>differentialAbundance-methods.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (<code><a href="Maaslin2.html">MicrobiomeDB::Maaslin2</a></code>) and DESeq2 (<code>DESeqDataSetFromCollection</code>).</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,missingOrNULL,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,function</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,character</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>data</dt>
<dd><p>AbundanceData object</p></dd>


<dt>covariate</dt>
<dd><p>character vector giving the name of a metadata variable of interest. If this
variable has only two values, you do not need to provide functions for arguments <code>groupA</code> and <code>groupB</code>.</p></dd>


<dt>groupA</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupA. In the specific case of <code>covariate</code> being a categorical variable,
this can be a character vector of values to be included in groupA.</p></dd>


<dt>groupB</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupB. If not provided, groupB will be the complement of groupA. In the specific
case of <code>covariate</code> being a categorical variable, this can be a character vector of values to be included in groupB.</p></dd>


<dt>method</dt>
<dd><p>string defining the the differential abundance method. Accepted values are 'DESeq2' and 'Maaslin2'.
Default is 'Maaslin2', as 'DESeq2' only supports counts.</p></dd>


<dt>verbose</dt>
<dd><p>boolean indicating if timed logging is desired</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>ComputeResult object</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co">## a continuous variable</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>         <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.58508 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.662564 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.664555 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.738306 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.741997 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.742562 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.743971 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.756709 INFO::Formula for fixed effects: expr ~  breastfed_duration</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.757517 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.757987 INFO::Total samples in data: 2533</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.758477 INFO::Min samples required with min abundance for a feature not to be filtered: 253.300000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.774253 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.775275 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Candidatus.Arthromitus, Candidatus.Obscuribacter, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Faecalicoccus, Faecalitalea, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavobacterium, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Marvinbryantia, Massilia, Megamonas, Merdibacter, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Myxococcus, NK4A214.group, Nakamurella, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phaseolibacter, Phocea, Phyllobacterium, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudoflavonifractor, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tetragenococcus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.781734 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.782286 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.782779 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.830136 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.833235 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.890612 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.891428 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.895796 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.899752 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.903663 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.907648 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.911549 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.915454 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.919341 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.923233 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.927146 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.931019 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.934929 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.93884 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.942779 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.946685 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.950634 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.954566 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.958561 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.962529 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.966551 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.970531 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.974504 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.978501 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.982405 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.986259 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.990128 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.994013 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:23.997871 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.001725 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.005629 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.009524 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.013374 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.027836 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.031946 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.03584 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.039733 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.043633 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.047519 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.051432 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.055338 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.059208 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.063096 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.067002 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.070907 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.074872 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.078811 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.082735 INFO::Fitting model to feature number 47, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.086662 INFO::Fitting model to feature number 48, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.090575 INFO::Fitting model to feature number 49, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.094487 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.098328 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.102216 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.106096 INFO::Fitting model to feature number 53, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.10999 INFO::Fitting model to feature number 54, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.113877 INFO::Fitting model to feature number 55, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.117774 INFO::Fitting model to feature number 56, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.121662 INFO::Fitting model to feature number 57, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.125575 INFO::Fitting model to feature number 58, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.129456 INFO::Fitting model to feature number 59, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.133352 INFO::Fitting model to feature number 60, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.137244 INFO::Fitting model to feature number 61, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.141146 INFO::Fitting model to feature number 62, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.145051 INFO::Fitting model to feature number 63, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.148943 INFO::Fitting model to feature number 64, Phascolarctobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.152829 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.156758 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.160609 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.164474 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.168453 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.172347 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.176216 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.180108 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.183986 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.187836 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.191691 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.195571 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.199658 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.203553 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.207428 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.21129 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.215168 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.219069 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.222934 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.226834 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.230709 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.234605 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.238486 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.242378 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.246247 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.250122 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.253998 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.257895 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.261856 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.265794 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.269718 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.273649 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.277527 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.28143 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.285334 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.289215 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.293372 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.297346 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.301262 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.305175 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.309055 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.312949 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.3169 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.332871 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.33828 INFO::Writing filtered data to file /tmp/RtmpHAaPNP/maaslin17917b1d9cea/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.510403 INFO::Writing filtered, normalized data to file /tmp/RtmpHAaPNP/maaslin17917b1d9cea/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.685203 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpHAaPNP/maaslin17917b1d9cea/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:24.967058 INFO::Writing residuals to file /tmp/RtmpHAaPNP/maaslin17917b1d9cea/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.021877 INFO::Writing fitted values to file /tmp/RtmpHAaPNP/maaslin17917b1d9cea/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.041 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin17917b1d9cea/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.042841 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin17917b1d9cea/significant_results.tsv</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.044711 Completed method=Maaslin2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.048043 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  Maaslin2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## same variable, using DESeq2</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>             <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>             <span class="st">'16S (V4) Genus (Absolute taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'DESeq2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.094417 Integer values detected. Converting collection to AbsoluteAbundanceData</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.232342 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.348756 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:37:25.350714 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> converting counts to integer mode</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>some variables in design formula are characters, converting to factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> using pre-existing size factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> gene-wise dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> mean-dispersion relationship</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> final dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- replacing outliers and refitting for 267 genes</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- DESeq argument 'minReplicatesForReplace' = 7 </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- original counts are preserved in counts(dds)</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.31721 Completed method=DESeq2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.320278 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  DESeq2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 3 values, one of which we exclude</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Russia"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Finland"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.659664 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.663383 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.663894 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.665431 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.679189 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.680043 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.680516 INFO::Total samples in data: 2692</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.680969 INFO::Min samples required with min abundance for a feature not to be filtered: 269.200000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.697963 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.699124 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asticcacaulis, Aureimonas, Bacillus, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Marinilactibacillus, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylobacillus, Methylobacterium.Methylorubrum, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, Paenibacillus, Paeniclostridium, Paludicola, Papillibacter, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Piscinibacter, Planifilum, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinivibrio, Sulfurospirillum, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gastranaerophilales, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.705918 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.706493 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.706963 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.758029 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.760953 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.823279 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.824076 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.82865 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.832946 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.837111 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.841245 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.845361 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.849418 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.853493 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.857554 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.861707 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.865847 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.86992 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.873948 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.878052 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.882085 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.886153 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.89019 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.894258 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.898302 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.902425 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.906503 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.910575 INFO::Fitting model to feature number 22, Corynebacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.914656 INFO::Fitting model to feature number 23, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.918717 INFO::Fitting model to feature number 24, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.922771 INFO::Fitting model to feature number 25, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.926803 INFO::Fitting model to feature number 26, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.930834 INFO::Fitting model to feature number 27, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.934852 INFO::Fitting model to feature number 28, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.938921 INFO::Fitting model to feature number 29, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.942972 INFO::Fitting model to feature number 30, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.947014 INFO::Fitting model to feature number 31, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.951056 INFO::Fitting model to feature number 32, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.955072 INFO::Fitting model to feature number 33, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.959096 INFO::Fitting model to feature number 34, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.963132 INFO::Fitting model to feature number 35, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.967148 INFO::Fitting model to feature number 36, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.971206 INFO::Fitting model to feature number 37, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.975228 INFO::Fitting model to feature number 38, Gordonibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.97926 INFO::Fitting model to feature number 39, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.983291 INFO::Fitting model to feature number 40, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.987355 INFO::Fitting model to feature number 41, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.991345 INFO::Fitting model to feature number 42, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.995411 INFO::Fitting model to feature number 43, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:04.999453 INFO::Fitting model to feature number 44, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.003493 INFO::Fitting model to feature number 45, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.007518 INFO::Fitting model to feature number 46, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.011616 INFO::Fitting model to feature number 47, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.015612 INFO::Fitting model to feature number 48, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.019688 INFO::Fitting model to feature number 49, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.023708 INFO::Fitting model to feature number 50, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.027813 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.031845 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.035898 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.039979 INFO::Fitting model to feature number 54, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.044064 INFO::Fitting model to feature number 55, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.048124 INFO::Fitting model to feature number 56, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.05223 INFO::Fitting model to feature number 57, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.056265 INFO::Fitting model to feature number 58, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.060311 INFO::Fitting model to feature number 59, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.064327 INFO::Fitting model to feature number 60, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.068392 INFO::Fitting model to feature number 61, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.072507 INFO::Fitting model to feature number 62, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.07675 INFO::Fitting model to feature number 63, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.081044 INFO::Fitting model to feature number 64, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.085203 INFO::Fitting model to feature number 65, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.089291 INFO::Fitting model to feature number 66, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.093391 INFO::Fitting model to feature number 67, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.097447 INFO::Fitting model to feature number 68, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.101566 INFO::Fitting model to feature number 69, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.105672 INFO::Fitting model to feature number 70, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.109739 INFO::Fitting model to feature number 71, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.113791 INFO::Fitting model to feature number 72, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.117945 INFO::Fitting model to feature number 73, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.12216 INFO::Fitting model to feature number 74, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.12659 INFO::Fitting model to feature number 75, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.130796 INFO::Fitting model to feature number 76, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.134915 INFO::Fitting model to feature number 77, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.138977 INFO::Fitting model to feature number 78, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.143029 INFO::Fitting model to feature number 79, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.147112 INFO::Fitting model to feature number 80, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.151138 INFO::Fitting model to feature number 81, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.155175 INFO::Fitting model to feature number 82, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.159222 INFO::Fitting model to feature number 83, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.16327 INFO::Fitting model to feature number 84, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.167311 INFO::Fitting model to feature number 85, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.17137 INFO::Fitting model to feature number 86, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.175401 INFO::Fitting model to feature number 87, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.179455 INFO::Fitting model to feature number 88, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.183465 INFO::Fitting model to feature number 89, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.187522 INFO::Fitting model to feature number 90, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.191538 INFO::Fitting model to feature number 91, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.195579 INFO::Fitting model to feature number 92, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.199598 INFO::Fitting model to feature number 93, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.219311 INFO::Fitting model to feature number 94, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.22369 INFO::Fitting model to feature number 95, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.227784 INFO::Fitting model to feature number 96, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.231911 INFO::Fitting model to feature number 97, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.235982 INFO::Fitting model to feature number 98, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.24007 INFO::Fitting model to feature number 99, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.244131 INFO::Fitting model to feature number 100, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.248236 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.252336 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.256462 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.260576 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.264654 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.268752 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.272855 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.288086 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.293267 INFO::Writing filtered data to file /tmp/RtmpHAaPNP/maaslin179113c84db7/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.483512 INFO::Writing filtered, normalized data to file /tmp/RtmpHAaPNP/maaslin179113c84db7/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.668782 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpHAaPNP/maaslin179113c84db7/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:05.969216 INFO::Writing residuals to file /tmp/RtmpHAaPNP/maaslin179113c84db7/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.030167 INFO::Writing fitted values to file /tmp/RtmpHAaPNP/maaslin179113c84db7/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.05044 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin179113c84db7/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.052192 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin179113c84db7/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## this case of categorical variables also has a 'short cut'</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="st">"Russia"</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Finland"</span>,<span class="st">"Estonia"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.372432 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.376118 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.376621 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.378494 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.393931 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.394781 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.395259 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.395714 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.430548 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.431822 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.439463 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.439995 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.440452 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.503737 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.506729 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.583119 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.583934 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.589081 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.593697 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.598292 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.602868 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.60744 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.612029 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.616551 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.621074 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.625619 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.630184 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.634748 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.639273 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.64377 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.648267 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.652768 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.657223 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.661733 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.666233 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.670834 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.675346 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.679826 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.684323 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.688808 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.693316 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.69784 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.702387 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.706885 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.711415 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.715898 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.720409 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.7249 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.729366 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.733864 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.738477 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.743015 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.747533 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.752084 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.756557 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.761047 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.765524 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.770056 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.774581 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.779089 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.783595 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.788087 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.792608 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.797227 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.801718 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.806378 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.810847 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.815366 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.819874 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.824356 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.828864 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.833393 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.837889 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.8424 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.846908 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.86628 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.87117 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.875799 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.880415 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.88501 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.889637 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.894264 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.89891 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.903604 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.908257 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.912905 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.917522 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.922175 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.926877 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.931514 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.936209 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.940846 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.945484 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.950098 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.954732 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.959324 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.963833 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.968351 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.97286 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.977495 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.982029 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.986594 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.991181 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:06.995723 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.000254 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.004778 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.009309 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.013786 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.018294 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.022851 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.027385 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.031974 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.036452 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.041032 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.045538 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.050066 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.054594 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.059106 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.063641 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.068154 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.072692 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.077233 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.08179 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.097786 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.1032 INFO::Writing filtered data to file /tmp/RtmpHAaPNP/maaslin17911fbce433/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.316844 INFO::Writing filtered, normalized data to file /tmp/RtmpHAaPNP/maaslin17911fbce433/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.536362 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpHAaPNP/maaslin17911fbce433/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.892767 INFO::Writing residuals to file /tmp/RtmpHAaPNP/maaslin17911fbce433/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.963108 INFO::Writing fitted values to file /tmp/RtmpHAaPNP/maaslin17911fbce433/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.986931 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin17911fbce433/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:07.988697 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin17911fbce433/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 2 values</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       <span class="st">"delivery_mode"</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span> </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.315604 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.319376 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.319889 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.32177 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.337501 INFO::Formula for fixed effects: expr ~  delivery_mode</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.338517 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.339014 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.339488 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.35915 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.36026 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.367819 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.368361 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.368827 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.44558 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.448792 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.526129 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.526971 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.532154 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.537199 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.5419 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.546536 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.551152 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.555733 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.560271 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.564803 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.569347 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.573877 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.578433 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.582984 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.587538 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.592106 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.59677 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.601325 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.605878 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.610446 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.614987 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.619528 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.624061 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.628619 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.633196 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.637746 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.642273 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.646876 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.651444 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.655983 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.660541 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.665087 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.669649 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.674165 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.678698 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.683222 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.687721 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.692201 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.696743 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.701252 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.705769 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.710282 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.714862 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.71939 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.72393 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.728455 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.732989 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.737519 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.742047 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.746619 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.751133 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.755612 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.760104 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.764632 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.769096 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.773574 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.778094 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.782607 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.787097 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.791589 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.796162 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.800681 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.805182 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.809662 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.814166 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.818722 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.82327 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.827848 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.832479 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.83697 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.841466 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.845975 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.850531 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.855004 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.859482 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.864071 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.868544 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.87304 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:08.87756 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.62363 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.628668 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.633278 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.637816 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.642293 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.646811 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.651381 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.655926 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.660468 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.664994 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.669616 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.674178 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.678753 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.683245 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.687698 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.692197 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.696699 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.701186 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.705679 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.710169 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.714745 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.719279 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.723782 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.728294 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.732804 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.7373 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.741783 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.746245 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.750755 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.766845 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.772072 INFO::Writing filtered data to file /tmp/RtmpHAaPNP/maaslin179163ce915c/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:09.985902 INFO::Writing filtered, normalized data to file /tmp/RtmpHAaPNP/maaslin179163ce915c/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:10.204228 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpHAaPNP/maaslin179163ce915c/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:10.561555 INFO::Writing residuals to file /tmp/RtmpHAaPNP/maaslin179163ce915c/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:10.632129 INFO::Writing fitted values to file /tmp/RtmpHAaPNP/maaslin179163ce915c/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:10.65704 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin179163ce915c/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-29 17:38:10.658779 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpHAaPNP/maaslin179163ce915c/significant_results.tsv</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

