<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><title>Differential abundance — differentialAbundance • MicrobiomeDB</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Differential abundance — differentialAbundance"><meta property="og:description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.2</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Differential abundance</h1>
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/R/method-differentialAbundance.R" class="external-link"><code>R/method-differentialAbundance.R</code></a></small>
      <div class="d-none name"><code>differentialAbundance-methods.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (<code><a href="Maaslin2.html">MicrobiomeDB::Maaslin2</a></code>) and DESeq2 (<code>DESeqDataSetFromCollection</code>).</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,missingOrNULL,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,function</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,character</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>data</dt>
<dd><p>AbundanceData object</p></dd>


<dt>covariate</dt>
<dd><p>character vector giving the name of a metadata variable of interest. If this
variable has only two values, you do not need to provide functions for arguments <code>groupA</code> and <code>groupB</code>.</p></dd>


<dt>groupA</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupA. In the specific case of <code>covariate</code> being a categorical variable,
this can be a character vector of values to be included in groupA.</p></dd>


<dt>groupB</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupB. If not provided, groupB will be the complement of groupA. In the specific
case of <code>covariate</code> being a categorical variable, this can be a character vector of values to be included in groupB.</p></dd>


<dt>method</dt>
<dd><p>string defining the the differential abundance method. Accepted values are 'DESeq2' and 'Maaslin2'.
Default is 'Maaslin2', as 'DESeq2' only supports counts.</p></dd>


<dt>verbose</dt>
<dd><p>boolean indicating if timed logging is desired</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>ComputeResult object</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co">## a continuous variable</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>         <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.726193 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.806644 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.80861 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.883354 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.887129 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.887631 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.888979 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.901567 INFO::Formula for fixed effects: expr ~  breastfed_duration</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.902339 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.902814 INFO::Total samples in data: 2533</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.903264 INFO::Min samples required with min abundance for a feature not to be filtered: 253.300000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.91913 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.920103 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Candidatus.Arthromitus, Candidatus.Obscuribacter, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Faecalicoccus, Faecalitalea, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavobacterium, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Marvinbryantia, Massilia, Megamonas, Merdibacter, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Myxococcus, NK4A214.group, Nakamurella, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phaseolibacter, Phocea, Phyllobacterium, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudoflavonifractor, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tetragenococcus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.926628 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.927181 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.927654 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.975251 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:20.978178 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.035944 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.036611 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.040939 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.044886 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.048899 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.052808 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.056714 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.060568 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.064498 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.068419 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.072305 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.076185 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.080067 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.08404 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.087916 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.091769 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.095672 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.099563 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.10345 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.107334 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.11123 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.115074 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.118957 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.122838 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.126762 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.130646 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.13454 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.138427 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.142317 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.146178 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.150073 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.153933 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.157817 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.172279 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.176307 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.180196 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.184105 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.187999 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.191912 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.195811 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.19969 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.203649 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.207554 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.211468 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.215368 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.219254 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.223128 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.227015 INFO::Fitting model to feature number 47, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.230928 INFO::Fitting model to feature number 48, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.23484 INFO::Fitting model to feature number 49, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.238744 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.242612 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.246516 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.250405 INFO::Fitting model to feature number 53, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.254327 INFO::Fitting model to feature number 54, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.258258 INFO::Fitting model to feature number 55, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.262169 INFO::Fitting model to feature number 56, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.266084 INFO::Fitting model to feature number 57, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.270016 INFO::Fitting model to feature number 58, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.273899 INFO::Fitting model to feature number 59, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.277796 INFO::Fitting model to feature number 60, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.281679 INFO::Fitting model to feature number 61, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.285597 INFO::Fitting model to feature number 62, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.289476 INFO::Fitting model to feature number 63, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.293365 INFO::Fitting model to feature number 64, Phascolarctobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.297195 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.301113 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.305038 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.308903 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.312816 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.316693 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.320673 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.324587 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.328495 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.332361 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.336258 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.340117 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.34402 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.347892 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.351789 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.355636 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.359573 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.363448 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.367361 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.371267 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.37515 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.379042 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.382943 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.386827 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.390754 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.394611 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.398517 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.402389 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.406271 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.410165 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.414053 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.41794 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.421848 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.425765 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.429658 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.433527 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.437441 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.441345 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.445215 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.44911 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.453008 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.456923 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.460817 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.476281 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.481391 INFO::Writing filtered data to file /tmp/RtmpcpLDsq/maaslin182958f20562/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.651259 INFO::Writing filtered, normalized data to file /tmp/RtmpcpLDsq/maaslin182958f20562/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:21.825479 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpcpLDsq/maaslin182958f20562/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.105976 INFO::Writing residuals to file /tmp/RtmpcpLDsq/maaslin182958f20562/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.160823 INFO::Writing fitted values to file /tmp/RtmpcpLDsq/maaslin182958f20562/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.179891 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin182958f20562/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.181747 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin182958f20562/significant_results.tsv</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.183631 Completed method=Maaslin2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.186919 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  Maaslin2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## same variable, using DESeq2</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>             <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>             <span class="st">'16S (V4) Genus (Absolute taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'DESeq2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.233707 Integer values detected. Converting collection to AbsoluteAbundanceData</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.372921 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.491191 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:05:22.493082 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> converting counts to integer mode</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>some variables in design formula are characters, converting to factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> using pre-existing size factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> gene-wise dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> mean-dispersion relationship</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> final dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- replacing outliers and refitting for 267 genes</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- DESeq argument 'minReplicatesForReplace' = 7 </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- original counts are preserved in counts(dds)</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.452442 Completed method=DESeq2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.455292 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  DESeq2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 3 values, one of which we exclude</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Russia"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Finland"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.788987 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.792745 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.793296 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.794847 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.808467 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.80928 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.809777 INFO::Total samples in data: 2692</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.810237 INFO::Min samples required with min abundance for a feature not to be filtered: 269.200000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.827302 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.828393 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asticcacaulis, Aureimonas, Bacillus, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Marinilactibacillus, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylobacillus, Methylobacterium.Methylorubrum, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, Paenibacillus, Paeniclostridium, Paludicola, Papillibacter, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Piscinibacter, Planifilum, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinivibrio, Sulfurospirillum, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gastranaerophilales, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.835203 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.835744 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.836215 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.887217 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.890117 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.953249 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.95392 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.958406 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.962484 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.966537 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.970565 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.974656 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.97877 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.982824 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.986851 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.990941 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.995013 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:01.99906 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.003085 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.007111 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.011143 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.015202 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.019245 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.023319 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.027363 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.031395 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.035461 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.039545 INFO::Fitting model to feature number 22, Corynebacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.043617 INFO::Fitting model to feature number 23, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.047695 INFO::Fitting model to feature number 24, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.051742 INFO::Fitting model to feature number 25, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.055789 INFO::Fitting model to feature number 26, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.059831 INFO::Fitting model to feature number 27, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.063855 INFO::Fitting model to feature number 28, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.067977 INFO::Fitting model to feature number 29, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.072065 INFO::Fitting model to feature number 30, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.076116 INFO::Fitting model to feature number 31, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.08022 INFO::Fitting model to feature number 32, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.084247 INFO::Fitting model to feature number 33, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.088283 INFO::Fitting model to feature number 34, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.092311 INFO::Fitting model to feature number 35, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.096361 INFO::Fitting model to feature number 36, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.100414 INFO::Fitting model to feature number 37, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.104427 INFO::Fitting model to feature number 38, Gordonibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.108458 INFO::Fitting model to feature number 39, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.112506 INFO::Fitting model to feature number 40, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.11653 INFO::Fitting model to feature number 41, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.120581 INFO::Fitting model to feature number 42, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.124613 INFO::Fitting model to feature number 43, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.128686 INFO::Fitting model to feature number 44, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.132739 INFO::Fitting model to feature number 45, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.136787 INFO::Fitting model to feature number 46, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.140812 INFO::Fitting model to feature number 47, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.14486 INFO::Fitting model to feature number 48, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.14893 INFO::Fitting model to feature number 49, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.152944 INFO::Fitting model to feature number 50, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.156949 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.160999 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.165031 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.169069 INFO::Fitting model to feature number 54, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.173072 INFO::Fitting model to feature number 55, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.17712 INFO::Fitting model to feature number 56, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.181146 INFO::Fitting model to feature number 57, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.18517 INFO::Fitting model to feature number 58, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.189183 INFO::Fitting model to feature number 59, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.19322 INFO::Fitting model to feature number 60, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.19732 INFO::Fitting model to feature number 61, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.201349 INFO::Fitting model to feature number 62, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.205521 INFO::Fitting model to feature number 63, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.209558 INFO::Fitting model to feature number 64, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.213614 INFO::Fitting model to feature number 65, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.21775 INFO::Fitting model to feature number 66, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.221814 INFO::Fitting model to feature number 67, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.225898 INFO::Fitting model to feature number 68, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.229973 INFO::Fitting model to feature number 69, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.234008 INFO::Fitting model to feature number 70, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.238068 INFO::Fitting model to feature number 71, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.242089 INFO::Fitting model to feature number 72, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.246147 INFO::Fitting model to feature number 73, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.250195 INFO::Fitting model to feature number 74, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.254249 INFO::Fitting model to feature number 75, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.258336 INFO::Fitting model to feature number 76, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.264294 INFO::Fitting model to feature number 77, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.270356 INFO::Fitting model to feature number 78, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.276424 INFO::Fitting model to feature number 79, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.280956 INFO::Fitting model to feature number 80, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.2852 INFO::Fitting model to feature number 81, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.289477 INFO::Fitting model to feature number 82, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.293725 INFO::Fitting model to feature number 83, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.298021 INFO::Fitting model to feature number 84, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.302349 INFO::Fitting model to feature number 85, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.306616 INFO::Fitting model to feature number 86, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.310944 INFO::Fitting model to feature number 87, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.315296 INFO::Fitting model to feature number 88, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.319663 INFO::Fitting model to feature number 89, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.323896 INFO::Fitting model to feature number 90, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.328223 INFO::Fitting model to feature number 91, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.332514 INFO::Fitting model to feature number 92, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.336824 INFO::Fitting model to feature number 93, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.35745 INFO::Fitting model to feature number 94, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.361852 INFO::Fitting model to feature number 95, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.365949 INFO::Fitting model to feature number 96, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.370032 INFO::Fitting model to feature number 97, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.374115 INFO::Fitting model to feature number 98, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.378175 INFO::Fitting model to feature number 99, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.382225 INFO::Fitting model to feature number 100, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.386301 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.390437 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.394543 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.39864 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.402717 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.406821 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.410921 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.426134 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.431287 INFO::Writing filtered data to file /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.611614 INFO::Writing filtered, normalized data to file /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:02.806041 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.104379 INFO::Writing residuals to file /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.164348 INFO::Writing fitted values to file /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.184078 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.185778 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin18293ef7fa4d/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## this case of categorical variables also has a 'short cut'</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="st">"Russia"</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Finland"</span>,<span class="st">"Estonia"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.508316 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.512117 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.512634 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.514577 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.529923 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.530693 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.53117 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.531629 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.567161 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.568405 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.576228 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.576758 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.577214 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.640195 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.643125 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.724907 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.725759 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.730913 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.735565 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.740228 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.744859 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.749754 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.754337 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.758994 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.763602 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.768208 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.772793 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.777478 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.782034 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.786609 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.791188 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.795729 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.800258 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.804847 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.809406 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.813989 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.818568 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.823202 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.827782 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.832374 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.836954 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.841567 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.846135 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.850696 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.855234 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.859819 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.864376 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.868938 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.873545 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.878182 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.882757 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.887332 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.891932 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.896556 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.901132 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.905739 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.910359 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.915969 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.920594 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.925175 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.929764 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.934336 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.938945 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.943511 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.948078 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.952678 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.957254 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.961821 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.9664 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.970968 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.975573 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.980142 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.984745 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.989378 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:03.994015 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.013911 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.01877 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.023483 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.02812 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.032887 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.037555 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.042253 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.046893 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.05156 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.056184 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.060843 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.065548 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.070232 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.074859 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.079466 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.084135 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.088763 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.093491 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.09815 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.102772 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.107378 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.112067 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.116699 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.12135 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.125925 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.130575 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.135213 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.139852 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.144447 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.149107 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.153734 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.158326 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.162936 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.167535 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.172157 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.176764 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.181396 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.185981 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.190548 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.195156 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.199742 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.20437 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.208968 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.213622 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.21819 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.222775 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.227333 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.231929 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.248397 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.253834 INFO::Writing filtered data to file /tmp/RtmpcpLDsq/maaslin1829636c0ded/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.466654 INFO::Writing filtered, normalized data to file /tmp/RtmpcpLDsq/maaslin1829636c0ded/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:04.684516 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpcpLDsq/maaslin1829636c0ded/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.040833 INFO::Writing residuals to file /tmp/RtmpcpLDsq/maaslin1829636c0ded/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.110667 INFO::Writing fitted values to file /tmp/RtmpcpLDsq/maaslin1829636c0ded/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.133854 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin1829636c0ded/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.135534 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin1829636c0ded/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 2 values</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       <span class="st">"delivery_mode"</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span> </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.463621 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.467457 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.467987 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.469952 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.485418 INFO::Formula for fixed effects: expr ~  delivery_mode</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.486199 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.486681 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.487139 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.508067 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.509145 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.516757 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.517331 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.5178 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.595434 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.598692 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.676078 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.676814 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.681899 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.686573 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.691218 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.695807 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.700436 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.705101 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.709721 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.714302 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.718885 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.723461 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.728055 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.732623 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.737156 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.741747 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.746856 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.751489 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.756076 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.76068 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.7653 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.769861 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.774409 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.779013 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.783566 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.788129 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.792669 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.797172 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.801694 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.80621 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.810732 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.815249 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.819816 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.824333 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.828884 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.833416 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.837937 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.842461 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.847 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.85158 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.856097 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.860668 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.865257 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.869822 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.874378 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.878933 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.883441 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.887961 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.892492 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.897027 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.901628 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.906229 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.910781 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.915342 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.919858 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.924426 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.928978 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.933581 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.938153 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.942717 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.947301 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.951825 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.956432 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.961003 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.965622 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.970194 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.974739 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.979341 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.983884 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.988487 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.993095 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:05.997683 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.002324 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.006877 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.011451 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.016011 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.020594 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.025131 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.029763 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.050245 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.05577 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.060707 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.065821 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.070664 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.075409 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.080204 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.084898 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.089619 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.094337 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.098994 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.103634 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.108304 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.11295 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.117575 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.122157 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.126796 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.131409 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.136036 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.140658 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.145314 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.149914 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.154467 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.159073 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.16363 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.168211 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.172767 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.177355 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.181927 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.198435 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.203814 INFO::Writing filtered data to file /tmp/RtmpcpLDsq/maaslin18293c01624c/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.415963 INFO::Writing filtered, normalized data to file /tmp/RtmpcpLDsq/maaslin18293c01624c/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.634736 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpcpLDsq/maaslin18293c01624c/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:06.992495 INFO::Writing residuals to file /tmp/RtmpcpLDsq/maaslin18293c01624c/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:07.063056 INFO::Writing fitted values to file /tmp/RtmpcpLDsq/maaslin18293c01624c/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:07.08741 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin18293c01624c/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-05-30 16:06:07.089212 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpcpLDsq/maaslin18293c01624c/significant_results.tsv</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

