<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><title>Differential abundance — differentialAbundance • MicrobiomeDB</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Differential abundance — differentialAbundance"><meta property="og:description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.1.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
    <a class="dropdown-item" href="../articles/user-data.html">Importing Data into MicrobiomeDB</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Differential abundance</h1>
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/R/method-differentialAbundance.R" class="external-link"><code>R/method-differentialAbundance.R</code></a></small>
      <div class="d-none name"><code>differentialAbundance-methods.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (<code><a href="Maaslin2.html">MicrobiomeDB::Maaslin2</a></code>) and DESeq2 (<code>DESeqDataSetFromCollection</code>).</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,missingOrNULL,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,function</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,character</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>data</dt>
<dd><p>AbundanceData object</p></dd>


<dt>covariate</dt>
<dd><p>character vector giving the name of a metadata variable of interest. If this
variable has only two values, you do not need to provide functions for arguments <code>groupA</code> and <code>groupB</code>.</p></dd>


<dt>groupA</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupA. In the specific case of <code>covariate</code> being a categorical variable,
this can be a character vector of values to be included in groupA.</p></dd>


<dt>groupB</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupB. If not provided, groupB will be the complement of groupA. In the specific
case of <code>covariate</code> being a categorical variable, this can be a character vector of values to be included in groupB.</p></dd>


<dt>method</dt>
<dd><p>string defining the the differential abundance method. Accepted values are 'DESeq2' and 'Maaslin2'.
Default is 'Maaslin2', as 'DESeq2' only supports counts.</p></dd>


<dt>verbose</dt>
<dd><p>boolean indicating if timed logging is desired</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>ComputeResult object</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co">## a continuous variable</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>         <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.550818 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.631296 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.633206 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.711635 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.71541 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.715917 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.717322 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.739221 INFO::Formula for fixed effects: expr ~  breastfed_duration</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.740184 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.74067 INFO::Total samples in data: 2533</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.741132 INFO::Min samples required with min abundance for a feature not to be filtered: 253.300000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.757937 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.758939 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Candidatus.Arthromitus, Candidatus.Obscuribacter, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Faecalicoccus, Faecalitalea, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavobacterium, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Marvinbryantia, Massilia, Megamonas, Merdibacter, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Myxococcus, NK4A214.group, Nakamurella, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phaseolibacter, Phocea, Phyllobacterium, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudoflavonifractor, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tetragenococcus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.765542 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.766178 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.766674 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.814735 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.817842 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.876111 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.876965 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.881374 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.885357 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.889344 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.893282 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.897257 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.90114 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.905063 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.908962 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.912866 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.916738 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.920661 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.924525 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.928438 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.932306 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.93621 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.940061 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.943947 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.947821 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.951705 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.955629 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.959736 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.963859 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.967915 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.971921 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.975951 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.979921 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.983861 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.987779 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.991728 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.99568 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:25.999617 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.003483 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.00743 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.011327 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.015246 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.019134 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.02336 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.027428 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.031322 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.035303 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.039267 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.043165 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.047057 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.050984 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.054963 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.058991 INFO::Fitting model to feature number 47, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.062933 INFO::Fitting model to feature number 48, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.066862 INFO::Fitting model to feature number 49, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.070733 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.074747 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.078698 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.082596 INFO::Fitting model to feature number 53, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.08647 INFO::Fitting model to feature number 54, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.09041 INFO::Fitting model to feature number 55, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.094269 INFO::Fitting model to feature number 56, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.098131 INFO::Fitting model to feature number 57, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.101981 INFO::Fitting model to feature number 58, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.105857 INFO::Fitting model to feature number 59, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.109653 INFO::Fitting model to feature number 60, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.113518 INFO::Fitting model to feature number 61, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.117414 INFO::Fitting model to feature number 62, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.121272 INFO::Fitting model to feature number 63, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.135796 INFO::Fitting model to feature number 64, Phascolarctobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.140525 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.144497 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.148461 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.152528 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.156587 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.160702 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.164821 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.169016 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.173388 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.177582 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.181638 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.185654 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.189679 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.19377 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.197909 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.201895 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.205831 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.209832 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.213809 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.217768 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.221688 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.225599 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.229501 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.233385 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.237308 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.241207 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.245156 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.24906 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.253023 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.256986 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.260983 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.264926 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.268841 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.272722 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.276747 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.280641 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.284543 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.288456 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.292368 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.296223 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.30011 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.30403 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.308004 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.323752 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.329064 INFO::Writing filtered data to file /tmp/RtmpKvXljO/maaslin17f82a98a169/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.502489 INFO::Writing filtered, normalized data to file /tmp/RtmpKvXljO/maaslin17f82a98a169/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.67993 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpKvXljO/maaslin17f82a98a169/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:26.963365 INFO::Writing residuals to file /tmp/RtmpKvXljO/maaslin17f82a98a169/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.018285 INFO::Writing fitted values to file /tmp/RtmpKvXljO/maaslin17f82a98a169/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.037839 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f82a98a169/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.04024 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f82a98a169/significant_results.tsv</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.042473 Completed method=Maaslin2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.0459 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  Maaslin2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## same variable, using DESeq2</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>             <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>             <span class="st">'16S (V4) Genus (Absolute taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'DESeq2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.092439 Integer values detected. Converting collection to AbsoluteAbundanceData</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.241791 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.372965 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:06:27.374849 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> converting counts to integer mode</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>some variables in design formula are characters, converting to factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> using pre-existing size factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> gene-wise dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> mean-dispersion relationship</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> final dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- replacing outliers and refitting for 267 genes</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- DESeq argument 'minReplicatesForReplace' = 7 </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- original counts are preserved in counts(dds)</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.451789 Completed method=DESeq2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.454952 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  DESeq2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 3 values, one of which we exclude</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Russia"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Finland"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.753267 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.756973 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.757476 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.75897 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.772289 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.773256 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.773758 INFO::Total samples in data: 2692</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.774216 INFO::Min samples required with min abundance for a feature not to be filtered: 269.200000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.802531 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.80383 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asticcacaulis, Aureimonas, Bacillus, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Marinilactibacillus, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylobacillus, Methylobacterium.Methylorubrum, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, Paenibacillus, Paeniclostridium, Paludicola, Papillibacter, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Piscinibacter, Planifilum, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinivibrio, Sulfurospirillum, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gastranaerophilales, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.810659 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.811214 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.811669 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.863018 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.866137 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.927976 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.928826 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.933372 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.937625 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.941827 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.946024 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.950161 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.954249 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.958308 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.962414 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.96647 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.970746 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.974772 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.978847 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.982871 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.987019 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.991053 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.995032 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:07.999082 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.003129 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.007142 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.011151 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.015173 INFO::Fitting model to feature number 22, Corynebacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.019178 INFO::Fitting model to feature number 23, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.023177 INFO::Fitting model to feature number 24, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.027234 INFO::Fitting model to feature number 25, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.031243 INFO::Fitting model to feature number 26, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.035231 INFO::Fitting model to feature number 27, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.039249 INFO::Fitting model to feature number 28, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.043281 INFO::Fitting model to feature number 29, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.047303 INFO::Fitting model to feature number 30, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.051318 INFO::Fitting model to feature number 31, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.055366 INFO::Fitting model to feature number 32, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.059395 INFO::Fitting model to feature number 33, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.063447 INFO::Fitting model to feature number 34, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.067494 INFO::Fitting model to feature number 35, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.071513 INFO::Fitting model to feature number 36, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.075527 INFO::Fitting model to feature number 37, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.079527 INFO::Fitting model to feature number 38, Gordonibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.083563 INFO::Fitting model to feature number 39, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.087634 INFO::Fitting model to feature number 40, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.091652 INFO::Fitting model to feature number 41, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.095671 INFO::Fitting model to feature number 42, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.099643 INFO::Fitting model to feature number 43, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.103634 INFO::Fitting model to feature number 44, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.107639 INFO::Fitting model to feature number 45, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.111789 INFO::Fitting model to feature number 46, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.115848 INFO::Fitting model to feature number 47, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.119838 INFO::Fitting model to feature number 48, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.123828 INFO::Fitting model to feature number 49, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.127794 INFO::Fitting model to feature number 50, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.131838 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.135888 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.139889 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.143858 INFO::Fitting model to feature number 54, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.147822 INFO::Fitting model to feature number 55, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.151766 INFO::Fitting model to feature number 56, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.155751 INFO::Fitting model to feature number 57, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.159724 INFO::Fitting model to feature number 58, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.163722 INFO::Fitting model to feature number 59, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.167707 INFO::Fitting model to feature number 60, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.171665 INFO::Fitting model to feature number 61, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.175687 INFO::Fitting model to feature number 62, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.17965 INFO::Fitting model to feature number 63, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.198344 INFO::Fitting model to feature number 64, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.20293 INFO::Fitting model to feature number 65, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.207152 INFO::Fitting model to feature number 66, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.21134 INFO::Fitting model to feature number 67, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.215471 INFO::Fitting model to feature number 68, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.219583 INFO::Fitting model to feature number 69, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.22364 INFO::Fitting model to feature number 70, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.227702 INFO::Fitting model to feature number 71, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.231804 INFO::Fitting model to feature number 72, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.235924 INFO::Fitting model to feature number 73, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.239993 INFO::Fitting model to feature number 74, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.244045 INFO::Fitting model to feature number 75, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.248075 INFO::Fitting model to feature number 76, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.252104 INFO::Fitting model to feature number 77, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.256132 INFO::Fitting model to feature number 78, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.260161 INFO::Fitting model to feature number 79, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.264172 INFO::Fitting model to feature number 80, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.268201 INFO::Fitting model to feature number 81, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.272193 INFO::Fitting model to feature number 82, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.276337 INFO::Fitting model to feature number 83, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.280385 INFO::Fitting model to feature number 84, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.284424 INFO::Fitting model to feature number 85, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.288443 INFO::Fitting model to feature number 86, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.292536 INFO::Fitting model to feature number 87, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.296566 INFO::Fitting model to feature number 88, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.300691 INFO::Fitting model to feature number 89, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.304762 INFO::Fitting model to feature number 90, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.30881 INFO::Fitting model to feature number 91, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.312885 INFO::Fitting model to feature number 92, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.316926 INFO::Fitting model to feature number 93, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.321012 INFO::Fitting model to feature number 94, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.325131 INFO::Fitting model to feature number 95, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.329287 INFO::Fitting model to feature number 96, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.333415 INFO::Fitting model to feature number 97, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.337539 INFO::Fitting model to feature number 98, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.341682 INFO::Fitting model to feature number 99, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.34579 INFO::Fitting model to feature number 100, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.349853 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.353918 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.357995 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.362001 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.366028 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.370045 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.374099 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.389577 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.394983 INFO::Writing filtered data to file /tmp/RtmpKvXljO/maaslin17f822a8088c/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.579106 INFO::Writing filtered, normalized data to file /tmp/RtmpKvXljO/maaslin17f822a8088c/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:08.767399 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpKvXljO/maaslin17f822a8088c/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:09.080816 INFO::Writing residuals to file /tmp/RtmpKvXljO/maaslin17f822a8088c/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:09.141061 INFO::Writing fitted values to file /tmp/RtmpKvXljO/maaslin17f822a8088c/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:09.161191 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f822a8088c/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:09.16315 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f822a8088c/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## this case of categorical variables also has a 'short cut'</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="st">"Russia"</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Finland"</span>,<span class="st">"Estonia"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.309951 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.313755 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.3143 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.316388 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.332054 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.333052 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.333529 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.334026 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.354587 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.35568 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.363358 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.363964 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.364421 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.42553 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.428698 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.502433 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.503265 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.508159 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.512649 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.517089 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.521461 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.525915 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.530358 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.534735 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.539099 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.543726 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.5482 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.552585 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.556953 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.561337 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.565676 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.570071 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.574441 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.578874 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.583282 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.587668 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.592059 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.596458 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.601008 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.605489 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.609987 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.614391 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.618743 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.623136 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.627652 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.632062 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.636482 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.640886 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.645382 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.64984 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.654253 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.65866 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.663043 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.667463 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.67185 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.676244 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.680776 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.685272 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.689746 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.694187 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.698608 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.702984 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.70745 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.711863 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.716437 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.721034 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.725595 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.730124 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.734526 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.738933 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.743325 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.747779 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.752236 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.756824 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.761376 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.765918 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.770387 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.774843 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.779272 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.783659 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.788054 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.79246 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.79687 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.801247 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.805631 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.810044 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.814429 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.818789 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.823162 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.827542 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.831936 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.83635 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.840774 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.845139 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.849588 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.854031 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.858435 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.862857 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.867305 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.871723 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.876204 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.880797 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.885275 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.889652 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.894063 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.898464 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.902846 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.907224 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.9116 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.91597 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.934312 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.93935 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.944013 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.948565 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.953038 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.957572 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.962137 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.966695 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.97119 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.975695 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.980193 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.984707 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:10.98928 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.005759 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.011444 INFO::Writing filtered data to file /tmp/RtmpKvXljO/maaslin17f86d89dc72/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.2294 INFO::Writing filtered, normalized data to file /tmp/RtmpKvXljO/maaslin17f86d89dc72/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.468891 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpKvXljO/maaslin17f86d89dc72/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.827232 INFO::Writing residuals to file /tmp/RtmpKvXljO/maaslin17f86d89dc72/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.897757 INFO::Writing fitted values to file /tmp/RtmpKvXljO/maaslin17f86d89dc72/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.921345 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f86d89dc72/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:11.923347 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f86d89dc72/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 2 values</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       <span class="st">"delivery_mode"</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span> </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.260851 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.264521 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.265021 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.266723 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.282416 INFO::Formula for fixed effects: expr ~  delivery_mode</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.283436 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.283917 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.284361 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.305823 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.306885 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.314624 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.315364 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.315823 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.377126 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.380236 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.455483 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.456357 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.461231 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.465843 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.470285 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.474709 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.47918 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.483567 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.488 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.492433 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.496893 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.501292 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.50573 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.510146 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.514527 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.518932 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.523377 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.527809 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.532265 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.536682 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.541164 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.545737 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.550231 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.554705 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.559182 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.563694 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.568157 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.572631 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.577031 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.581404 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.585844 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.590306 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.594726 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.599288 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.603791 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.608286 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.612728 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.61718 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.621696 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.626142 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.630545 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.634949 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.63938 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.643852 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.648276 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.652734 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.65712 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.661541 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.665993 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.670393 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.674795 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.679204 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.683626 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.688011 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.692371 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.696867 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.701344 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.705796 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.710218 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.714616 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.719058 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.723476 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.727872 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.732257 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.736686 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.741123 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.74552 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.763377 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.768376 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.773079 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.777784 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.782376 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.786921 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.791486 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.795988 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.800495 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.804965 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.809479 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.814002 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.818506 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.822958 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.827417 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.831888 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.836315 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.840858 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.845387 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.850167 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.85493 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.859844 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.864682 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.869353 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.873959 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.8787 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.883423 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.888017 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.892524 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.897049 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.901509 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.90599 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.910444 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.915098 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.919576 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.92402 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.928432 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.932854 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.93729 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.941679 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.946161 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.962534 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:12.967932 INFO::Writing filtered data to file /tmp/RtmpKvXljO/maaslin17f8bb07323/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:13.184663 INFO::Writing filtered, normalized data to file /tmp/RtmpKvXljO/maaslin17f8bb07323/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:13.40505 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpKvXljO/maaslin17f8bb07323/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:13.764286 INFO::Writing residuals to file /tmp/RtmpKvXljO/maaslin17f8bb07323/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:13.835 INFO::Writing fitted values to file /tmp/RtmpKvXljO/maaslin17f8bb07323/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:13.859495 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f8bb07323/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-26 13:07:13.861506 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpKvXljO/maaslin17f8bb07323/significant_results.tsv</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

