<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><title>Differential abundance — differentialAbundance • MicrobiomeDB</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Differential abundance — differentialAbundance"><meta property="og:description" content="This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (MicrobiomeDB::Maaslin2) and DESeq2 (DESeqDataSetFromCollection)."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.2</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
    <a class="dropdown-item" href="../articles/user-data.html">Importing Data into MicrobiomeDB</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Differential abundance</h1>
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/R/method-differentialAbundance.R" class="external-link"><code>R/method-differentialAbundance.R</code></a></small>
      <div class="d-none name"><code>differentialAbundance-methods.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>This function returns the fold change and associated p value for
a differential abundance analysis. It is useful for finding taxa with
an abundance that strongly differs between two groups of samples,
such as finding taxa with abundance that differs between skin and saliva samples.
This function allows one to recreate the results of differential abundance analysis
from MicrobiomeDB.org. However, we recognize that this function makes some assumptions
about the data which may not be valid in other contexts. For better support of
longitudinal studies or metabolomic data, for example, please see our wrapper/ helper methods
for Maaslin2 (<code><a href="Maaslin2.html">MicrobiomeDB::Maaslin2</a></code>) and DESeq2 (<code>DESeqDataSetFromCollection</code>).</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,missingOrNULL,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,missingOrNULL</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,function,function</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co"># S4 method for CollectionWithMetadata,character,character,character</span></span>
<span><span class="fu">differentialAbundance</span><span class="op">(</span></span>
<span>  <span class="va">data</span>,</span>
<span>  <span class="va">covariate</span>,</span>
<span>  <span class="va">groupA</span>,</span>
<span>  <span class="va">groupB</span>,</span>
<span>  method <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Maaslin2"</span>, <span class="st">"DESeq2"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="cn">TRUE</span>, <span class="cn">FALSE</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="arguments">Arguments<a class="anchor" aria-label="anchor" href="#arguments"></a></h2>
    <dl><dt>data</dt>
<dd><p>AbundanceData object</p></dd>


<dt>covariate</dt>
<dd><p>character vector giving the name of a metadata variable of interest. If this
variable has only two values, you do not need to provide functions for arguments <code>groupA</code> and <code>groupB</code>.</p></dd>


<dt>groupA</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupA. In the specific case of <code>covariate</code> being a categorical variable,
this can be a character vector of values to be included in groupA.</p></dd>


<dt>groupB</dt>
<dd><p>A function that takes a vector of values and returns TRUE or FALSE for each value. This will
be used to assign samples to groupB. If not provided, groupB will be the complement of groupA. In the specific
case of <code>covariate</code> being a categorical variable, this can be a character vector of values to be included in groupB.</p></dd>


<dt>method</dt>
<dd><p>string defining the the differential abundance method. Accepted values are 'DESeq2' and 'Maaslin2'.
Default is 'Maaslin2', as 'DESeq2' only supports counts.</p></dd>


<dt>verbose</dt>
<dd><p>boolean indicating if timed logging is desired</p></dd>

</dl></div>
    <div class="section level2">
    <h2 id="value">Value<a class="anchor" aria-label="anchor" href="#value"></a></h2>
    

<p>ComputeResult object</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co">## a continuous variable</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>         <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.276221 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.369234 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.371206 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.445819 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.449596 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.450109 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.451563 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.464439 INFO::Formula for fixed effects: expr ~  breastfed_duration</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.46522 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.465705 INFO::Total samples in data: 2533</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.466158 INFO::Min samples required with min abundance for a feature not to be filtered: 253.300000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.483089 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.484122 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Candidatus.Arthromitus, Candidatus.Obscuribacter, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Faecalicoccus, Faecalitalea, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavobacterium, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Marvinbryantia, Massilia, Megamonas, Merdibacter, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Myxococcus, NK4A214.group, Nakamurella, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phaseolibacter, Phocea, Phyllobacterium, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudoflavonifractor, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tetragenococcus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.490594 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.49114 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.491634 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.539182 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.542199 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.60563 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.606468 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.612157 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.617399 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.622536 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.627699 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.632823 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.637932 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.643048 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.64823 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.668236 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.673465 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.678574 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.683741 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.688849 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.694069 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.699145 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.704343 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.709505 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.714586 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.719723 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.724827 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.741209 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.745591 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.749552 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.753469 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.757408 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.761394 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.765327 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.769235 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.773186 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.777126 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.781031 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.784949 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.78892 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.79284 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.796813 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.800747 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.804694 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.808598 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.812525 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.81646 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.820418 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.82435 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.828273 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.832224 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.836208 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.840137 INFO::Fitting model to feature number 47, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.844076 INFO::Fitting model to feature number 48, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.84799 INFO::Fitting model to feature number 49, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.851926 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.855798 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.859801 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.863713 INFO::Fitting model to feature number 53, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.867625 INFO::Fitting model to feature number 54, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.871534 INFO::Fitting model to feature number 55, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.875386 INFO::Fitting model to feature number 56, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.879255 INFO::Fitting model to feature number 57, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.883164 INFO::Fitting model to feature number 58, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.887047 INFO::Fitting model to feature number 59, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.89095 INFO::Fitting model to feature number 60, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.894836 INFO::Fitting model to feature number 61, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.900163 INFO::Fitting model to feature number 62, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.904246 INFO::Fitting model to feature number 63, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.908199 INFO::Fitting model to feature number 64, Phascolarctobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.912095 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.916185 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.920273 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.924216 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.928131 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.932047 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.935972 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.939906 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.943814 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.94783 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.951738 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.955635 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.959548 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.963507 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.967406 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.971278 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.97522 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.979172 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.983067 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.986956 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.990868 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.994805 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:00.998712 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.003449 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.010453 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.016249 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.02018 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.024112 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.028043 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.031922 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.035815 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.039749 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.043732 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.047659 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.051605 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.055569 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.059593 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.063617 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.067568 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.071528 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.075475 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.07936 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.083257 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.099019 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.104257 INFO::Writing filtered data to file /tmp/RtmpSSLDrC/maaslin179679246c60/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.276196 INFO::Writing filtered, normalized data to file /tmp/RtmpSSLDrC/maaslin179679246c60/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.451525 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpSSLDrC/maaslin179679246c60/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.735842 INFO::Writing residuals to file /tmp/RtmpSSLDrC/maaslin179679246c60/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.791122 INFO::Writing fitted values to file /tmp/RtmpSSLDrC/maaslin179679246c60/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.810173 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin179679246c60/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.811977 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin179679246c60/significant_results.tsv</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.813893 Completed method=Maaslin2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.817315 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  Maaslin2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## same variable, using DESeq2</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>             <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>             <span class="st">'16S (V4) Genus (Absolute taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"breastfed_duration"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&lt;</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">&gt;=</span><span class="fl">300</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'DESeq2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:01.863737 Integer values detected. Converting collection to AbsoluteAbundanceData</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:02.00999 Found NAs in specified variable. Removing these records.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:02.131337 Received abundance table with 2533 samples and 674 taxa.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:02.133374 Found 2533 samples with a value for breastfed_duration in either groupA or groupB. The calculation will continue with only these samples.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> converting counts to integer mode</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>some variables in design formula are characters, converting to factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> using pre-existing size factors</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> gene-wise dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> mean-dispersion relationship</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> final dispersion estimates</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- replacing outliers and refitting for 267 genes</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- DESeq argument 'minReplicatesForReplace' = 7 </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> -- original counts are preserved in counts(dds)</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> estimating dispersions</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> fitting model and testing</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.54936 Completed method=DESeq2. Formatting results.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.552658 Differential abundance computation completed with parameters recordIdColumn =  16S_rRNA_(V4)_assay_Id , comparatorColName =  breastfed_duration , method =  DESeq2 , groupA = groupA , groupB =  groupB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 3 values, one of which we exclude</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Russia"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span><span class="va">x</span><span class="op">==</span><span class="st">"Finland"</span><span class="op">}</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.854447 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.858364 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.858887 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.860375 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.873858 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.874689 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.875173 INFO::Total samples in data: 2692</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.875674 INFO::Min samples required with min abundance for a feature not to be filtered: 269.200000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.892484 INFO::Total filtered features: 522</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.893606 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asticcacaulis, Aureimonas, Bacillus, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FCS020.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Marinilactibacillus, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylobacillus, Methylobacterium.Methylorubrum, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, Paenibacillus, Paeniclostridium, Paludicola, Papillibacter, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Piscinibacter, Planifilum, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinivibrio, Sulfurospirillum, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gastranaerophilales, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.900399 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.90095 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.901415 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.95261 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:41.955663 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.016924 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.017745 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.022382 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.026594 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.030837 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.034924 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.039017 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.043118 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.047162 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.051225 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.05529 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.0594 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.06348 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.067538 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.071602 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.075644 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.079688 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.083683 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.087746 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.091784 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.095822 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.099818 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.103877 INFO::Fitting model to feature number 22, Corynebacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.107884 INFO::Fitting model to feature number 23, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.111922 INFO::Fitting model to feature number 24, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.115915 INFO::Fitting model to feature number 25, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.119931 INFO::Fitting model to feature number 26, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.123943 INFO::Fitting model to feature number 27, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.127957 INFO::Fitting model to feature number 28, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.131979 INFO::Fitting model to feature number 29, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.136014 INFO::Fitting model to feature number 30, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.140028 INFO::Fitting model to feature number 31, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.144056 INFO::Fitting model to feature number 32, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.148062 INFO::Fitting model to feature number 33, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.152088 INFO::Fitting model to feature number 34, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.156105 INFO::Fitting model to feature number 35, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.160123 INFO::Fitting model to feature number 36, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.164107 INFO::Fitting model to feature number 37, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.168133 INFO::Fitting model to feature number 38, Gordonibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.172143 INFO::Fitting model to feature number 39, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.176163 INFO::Fitting model to feature number 40, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.180228 INFO::Fitting model to feature number 41, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.184272 INFO::Fitting model to feature number 42, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.188304 INFO::Fitting model to feature number 43, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.192372 INFO::Fitting model to feature number 44, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.196521 INFO::Fitting model to feature number 45, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.200573 INFO::Fitting model to feature number 46, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.204621 INFO::Fitting model to feature number 47, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.208643 INFO::Fitting model to feature number 48, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.212678 INFO::Fitting model to feature number 49, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.216693 INFO::Fitting model to feature number 50, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.237363 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.241775 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.245833 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.249891 INFO::Fitting model to feature number 54, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.253936 INFO::Fitting model to feature number 55, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.258089 INFO::Fitting model to feature number 56, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.262167 INFO::Fitting model to feature number 57, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.266228 INFO::Fitting model to feature number 58, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.270292 INFO::Fitting model to feature number 59, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.274391 INFO::Fitting model to feature number 60, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.278463 INFO::Fitting model to feature number 61, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.282557 INFO::Fitting model to feature number 62, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.286628 INFO::Fitting model to feature number 63, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.290773 INFO::Fitting model to feature number 64, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.294861 INFO::Fitting model to feature number 65, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.298933 INFO::Fitting model to feature number 66, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.303037 INFO::Fitting model to feature number 67, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.3071 INFO::Fitting model to feature number 68, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.311216 INFO::Fitting model to feature number 69, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.315327 INFO::Fitting model to feature number 70, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.319468 INFO::Fitting model to feature number 71, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.323576 INFO::Fitting model to feature number 72, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.327655 INFO::Fitting model to feature number 73, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.331716 INFO::Fitting model to feature number 74, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.335782 INFO::Fitting model to feature number 75, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.33989 INFO::Fitting model to feature number 76, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.34396 INFO::Fitting model to feature number 77, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.348025 INFO::Fitting model to feature number 78, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.352093 INFO::Fitting model to feature number 79, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.356213 INFO::Fitting model to feature number 80, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.360267 INFO::Fitting model to feature number 81, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.364417 INFO::Fitting model to feature number 82, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.368517 INFO::Fitting model to feature number 83, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.372602 INFO::Fitting model to feature number 84, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.376681 INFO::Fitting model to feature number 85, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.380747 INFO::Fitting model to feature number 86, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.38485 INFO::Fitting model to feature number 87, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.388945 INFO::Fitting model to feature number 88, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.393036 INFO::Fitting model to feature number 89, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.397102 INFO::Fitting model to feature number 90, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.401193 INFO::Fitting model to feature number 91, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.405245 INFO::Fitting model to feature number 92, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.409283 INFO::Fitting model to feature number 93, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.413393 INFO::Fitting model to feature number 94, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.417479 INFO::Fitting model to feature number 95, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.421574 INFO::Fitting model to feature number 96, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.425625 INFO::Fitting model to feature number 97, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.429742 INFO::Fitting model to feature number 98, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.433783 INFO::Fitting model to feature number 99, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.437838 INFO::Fitting model to feature number 100, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.441864 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.446011 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.45007 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.454147 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.458179 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.462241 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.466258 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.4816 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.48682 INFO::Writing filtered data to file /tmp/RtmpSSLDrC/maaslin17962d719ffd/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.681001 INFO::Writing filtered, normalized data to file /tmp/RtmpSSLDrC/maaslin17962d719ffd/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:42.868374 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpSSLDrC/maaslin17962d719ffd/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.169962 INFO::Writing residuals to file /tmp/RtmpSSLDrC/maaslin17962d719ffd/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.230861 INFO::Writing fitted values to file /tmp/RtmpSSLDrC/maaslin17962d719ffd/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.251006 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin17962d719ffd/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.252996 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin17962d719ffd/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## this case of categorical variables also has a 'short cut'</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>, </span></span>
<span class="r-in"><span>       <span class="st">"country"</span>, </span></span>
<span class="r-in"><span>       groupA <span class="op">=</span> <span class="st">"Russia"</span>,</span></span>
<span class="r-in"><span>       groupB <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Finland"</span>,<span class="st">"Estonia"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.581468 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.58525 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.585758 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.587509 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.603157 INFO::Formula for fixed effects: expr ~  country</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.604011 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.604497 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.604963 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.625586 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.626761 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.634395 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.634973 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.635481 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.697929 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.701053 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.775866 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.776657 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.781664 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.786234 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.790787 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.795278 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.799824 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.804371 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.808853 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.813342 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.817789 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.822287 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.826771 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.831265 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.835797 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.840242 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.844744 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.849184 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.86941 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.874161 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.878676 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.883235 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.8878 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.892359 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.89691 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.901498 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.90606 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.910612 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.915197 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.919803 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.92435 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.92893 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.933459 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.93801 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.942563 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.947131 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.951724 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.956257 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.960813 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.965354 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.969896 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.97437 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.97887 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.983405 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.987925 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.992429 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:43.99695 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.001505 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.005985 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.010508 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.014986 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.019506 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.024029 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.028512 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.033036 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.037563 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.042112 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.046641 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.051173 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.055761 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.060266 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.064806 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.069317 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.073812 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.078304 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.082811 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.087282 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.091802 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.096375 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.101027 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.10564 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.110204 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.114686 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.119157 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.123703 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.128226 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.132691 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.137176 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.14168 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.146139 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.150592 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.15504 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.159581 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.164048 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.168549 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.173024 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.177519 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.181986 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.186492 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.205783 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.210811 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.215383 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.219908 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.224488 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.229064 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.233708 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.238305 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.242946 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.247564 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.252196 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.256825 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.26141 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.265973 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.270511 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.275066 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.279636 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.284192 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.288757 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.304916 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.310348 INFO::Writing filtered data to file /tmp/RtmpSSLDrC/maaslin1796311c2a51/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.525466 INFO::Writing filtered, normalized data to file /tmp/RtmpSSLDrC/maaslin1796311c2a51/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:44.758857 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpSSLDrC/maaslin1796311c2a51/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:45.119312 INFO::Writing residuals to file /tmp/RtmpSSLDrC/maaslin1796311c2a51/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:45.190759 INFO::Writing fitted values to file /tmp/RtmpSSLDrC/maaslin1796311c2a51/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:45.214252 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin1796311c2a51/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:45.216207 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin1796311c2a51/significant_results.tsv</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## a categorical variable with 2 values</span></span></span>
<span class="r-in"><span><span class="va">diffAbundOutput</span> <span class="op">&lt;-</span> <span class="fu">MicrobiomeDB</span><span class="fu">::</span><span class="fu">differentialAbundance</span><span class="op">(</span></span></span>
<span class="r-in"><span>       <span class="fu"><a href="getCollection.html">getCollection</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/DiabImmune.html" class="external-link">DiabImmune</a></span>, </span></span>
<span class="r-in"><span>  <span class="st">'16S (V4) Genus (Relative taxonomic abundance analysis)'</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>       <span class="st">"delivery_mode"</span>,</span></span>
<span class="r-in"><span>       method<span class="op">=</span><span class="st">'Maaslin2'</span>, </span></span>
<span class="r-in"><span>       verbose<span class="op">=</span><span class="cn">FALSE</span></span></span>
<span class="r-in"><span><span class="op">)</span> </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output feature tables folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "Creating output fits folder"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.375226 INFO::Writing function arguments to log file</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.378945 INFO::Verifying options selected are valid</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.379491 INFO::Determining format of input files</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.381464 INFO::Input format is data samples as rows and metadata samples as rows</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.396833 INFO::Formula for fixed effects: expr ~  delivery_mode</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.397663 INFO::Filter data based on min abundance and min prevalence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.398137 INFO::Total samples in data: 3184</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.39858 INFO::Min samples required with min abundance for a feature not to be filtered: 318.400000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.418689 INFO::Total filtered features: 564</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.419746 INFO::Filtered feature names from abundance and prevalence filtering: X28.4, Abiotrophia, Acetanaerobacterium, Acetobacter, Achromobacter, Acidaminococcus, Acidibacter, Acidiphilium, Acidipropionibacterium, Acidocella, Acidovorax, Acinetobacter, Actinobacillus, Actinobaculum, Actinotignum, Adlercreutzia, Aerococcus, Aeromonas, Afipia, Aggregatibacter, Alcaligenes, Alicycliphilus, Allisonella, Allobaculum, Alloprevotella, Allorhizobium.Neorhizobium.Pararhizobium.Rhizobium, Alloscardovia, Amaricoccus, Amnibacterium, Anaerobium, Anaerocolumna, Anaerofilum, Anaerofustis, Anaeroglobus, Anaeromyxobacter, Anaeroplasma, Anaerosporobacter, Anaerosporomusa, Anaerostignum, Anaerotruncus, Angelakisella, Angustibacter, Anoxybacillus, Aquabacterium, Aquamonas, Arcanobacterium, Arcticibacter, Asaccharospora, Asaia, Asinibacterium, Asteroleplasma, Asticcacaulis, Aureimonas, Azospira, Bacillus, Bacteriovorax, Bdellovibrio, Belnapia, Bergeyella, Blastomonas, Bordetella, Bosea, Brachybacterium, Bradyrhizobium, Brevibacterium, Brevundimonas, Brucella, Bulleidia, Burkholderia.Caballeronia.Paraburkholderia, Buttiauxella, Butyricimonas, CAG.56, CAG.873, CHKCI001, CHKCI002, CL500.29.marine.group, Caloribacterium, Candidatus.Arthromitus, Candidatus.Metachlamydia, Candidatus.Obscuribacter, Candidatus.Phytoplasma, Candidatus.Soleaferrea, Candidatus.Stoquefichus, Capnocytophaga, Caproiciproducens, Cardiobacterium, Carnobacterium, Caryophanon, Castellaniella, Catabacter, Catenibacillus, Catenibacterium, Catenisphaera, Catonella, Caulobacter, Cellulomonas, Cellulosilyticum, Cellvibrio, Centipeda, Cetobacterium, Christensenella, Chryseobacterium, Chryseomicrobium, Chungangia, Citrobacter, Clavibacter, Cloacibacillus, Cloacibacterium, Clostridium.sensu.stricto.12, Clostridium.sensu.stricto.13, Clostridium.sensu.stricto.15, Clostridium.sensu.stricto.18, Clostridium.sensu.stricto.2, Clostridium.sensu.stricto.3, Clostridium.sensu.stricto.4, Clostridium.sensu.stricto.6, Comamonas, Conexibacter, Conservatibacter, Coprobacter, Coriobacteriaceae.UCG.002, Corynebacterium, Cosenzaea, Criibacterium, Cryptobacterium, Curvibacter, Cutibacterium, Cytophaga, DNF00809, DSSD61, Dechloromonas, Defluviicoccus, Defluviitaleaceae.UCG.011, Delftia, Denitrobacterium, Dermabacter, Desemzia, Desulfovibrio, Devosia, Diaphorobacter, Dickeya, Dielma, Dietzia, Dinghuibacter, Diplosphaera, Dolosigranulum, Duganella, Dysgonomonas, Edaphobaculum, Eikenella, Empedobacter, Endobacter, Enhydrobacter, Enterobacter, Enterorhabdus, Enteroscipio, Epulopiscium, Erwinia, Erysipelotrichaceae.UCG.006, Eubacterium, Exiguobacterium, Extensimonas, F0058, F0332, FD2005, Facklamia, Faecalicoccus, Faecalitalea, Falsochrobactrum, Family.XIII.UCG.001, Fictibacillus, Flaviflexus, Flavihumibacter, Flavobacterium, Floricoccus, Fluviicola, Fournierella, Fretibacterium, Frisingicoccus, Fructobacillus, GCA.900066575, GCA.900066755, Galbibacter, Garciella, Gemmata, Geobacillus, Gluconobacter, Glutamicibacter, Gordonia, Gordonibacter, Gulbenkiania, Hafnia.Obesumbacterium, Halomonas, Haoranjiania, Harryflintia, Helicobacter, Hephaestia, Herbaspirillum, Herbinix, Holdemanella, Howardella, Hydrocarboniphaga, Hydrogenophilus, Hymenobacter, Iamia, Ignavigranum, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Lachnospirales_Lachnospiraceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Erwiniaceae_Incertae_Sedis., Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Gammaproteobacteria_Enterobacterales_Morganellaceae_Incertae_Sedis., Janthinobacterium, Jeotgalicoccus, Johnsonella, Kingella, Knoellia, Kocuria, Kosakonia, Lachnoanaerobaculum, Lachnobacterium, Lachnospiraceae.AC2044.group, Lachnospiraceae.FE2018.group, Lachnospiraceae.NC2004.group, Lachnospiraceae.UCG.002, Lachnospiraceae.UCG.003, Lachnospiraceae.UCG.008, Lachnospiraceae.XPB1014.group, Lachnotalea, Lactonifactor, Lautropia, Lawsonella, Legionella, Leptotrichia, Leuconostoc, Libanicoccus, Luteimonas, Luteolibacter, Macellibacteroides, Macrococcus, Mailhella, Marinilactibacillus, Marmoricola, Martelella, Marvinbryantia, Massilia, Megamonas, Merdibacter, Mesorhizobium, Methanobrevibacter, Methylibium, Methylobacillus, Methylobacterium.Methylorubrum, Methylorosula, Methylotenera, Micrococcus, Mitsuokella, Mobiluncus, Mogibacterium, Moheibacter, Mongoliitalea, Moraxella, Morganella, Moryella, Mucilaginibacter, Muribaculum, Murimonas, Mycobacterium, Mycoplasma, Myxococcus, NK4A214.group, Nakamurella, Nannocystis, Necropsobacter, Negativibacillus, Negativicoccus, Neochlamydia, Neomegalonema, Neorhizobium, Neoscardovia, Nesterenkonia, Nocardioides, Nocardiopsis, Novosphingobium, Oceanivirga, Oceanobacillus, Ochrobactrum, Olsenella, Oribacterium, Ottowia, Oxalobacter, Oxobacter, P3OB.42, Paenibacillus, Paeniclostridium, Paenisporosarcina, Paludicola, Papillibacter, Paraclostridium, Paracoccus, Paramesorhizobium, Paraprevotella, Parapusillimonas, Parascardovia, Parvibacter, Pasteurella, Paucibacter, Pediococcus, Pedobacter, Peptoclostridium, Peptococcus, Peptostreptococcus, Perlucidibaca, Phascolarctobacterium, Phaseolibacter, Phocea, Phoenicibacter, Phyllobacterium, Piscinibacter, Planifilum, Plesiomonas, Plot4.2H12, Pluralibacter, Polaromonas, Porphyromonas, Prevotellaceae.NK3B31.group, Prevotellaceae.UCG.001, Propionibacterium, Propionimicrobium, Prosthecobacter, Proteus, Providencia, Pseudacidovorax, Pseudaminobacter, Pseudocitrobacter, Pseudoflavonifractor, Pseudoglutamicibacter, Pseudoramibacter, Pseudoxanthomonas, Psychrobacter, Pygmaiobacter, Pyramidobacter, Qipengyuania, Ralstonia, Raoultella, Raoultibacter, Reyranella, Rhizorhapis, Rhodanobacter, Rhodobacter, Rhodococcus, Rikenella, Rikenellaceae.RC9.gut.group, Rivicola, Robinsoniella, Rodentibacter, Roseisolibacter, Rubellimicrobium, Rummeliibacillus, S5.A14a, SH.PL14, Saccharothrix, Salinimicrobium, Salinisphaera, Salipaludibacillus, Salmonella, Sanguibacteroides, Sarcina, Scardovia, Sedimentibacter, Segetibacter, Selenomonas, Senegalimassilia, Serratia, Shimwellia, Shuttleworthia, Slackia, Sneathia, Solibacillus, Sphingoaurantiacus, Sphingobacterium, Sphingobium, Sphingomonas, Sphingopyxis, Sporolactobacillus, Stenotrophomonas, Stomatobaculum, Streptobacillus, Succinatimonas, Succiniclasticum, Succinivibrio, Sulfurospirillum, Synergistes, TM7x, Tannerella, Tatumella, Tepidimonas, Tepidiphilus, Tetragenococcus, Thermus, Timonella, Tolumonas, Trabulsiella, Tropheryma, Trueperella, UC5.1.2E3, UCG.004, UCG.007, UCG.008, UCG.009, Ureibacillus, Varibaculum, Variovorax, Verticiella, Victivallis, Vulcaniibacterium, Weissella, Xanthomonas, Yersinia, Yokenella, Zoogloea, X.Bacteroides..pectinophilus.group, X.Eubacterium..brachy.group, X.Eubacterium..nodatum.group, X.Eubacterium..oxidoreducens.group, X.Eubacterium..ruminantium.group, unclassified.0319.6G20, unclassified.67.14, unclassified.A0839, unclassified.Acetobacteraceae, unclassified.Acidaminococcaceae, unclassified.Actinobacteria, unclassified.Actinomycetaceae, unclassified.Aeromonadales, unclassified.Anaerococcus, unclassified.Anaerosalibacter, unclassified.Anaerovoracaceae, unclassified.Armatimonadales, unclassified.Atopobiaceae, unclassified.Bacillaceae, unclassified.Bacillales, unclassified.Bacilli, unclassified.Bacteroidales, unclassified.Bacteroidia, unclassified.Barnesiellaceae, unclassified.Bifidobacteriaceae, unclassified.Burkholderiales, unclassified.Butyricicoccaceae, unclassified.Caloramatoraceae, unclassified.Carnobacteriaceae, unclassified.Caulobacteraceae, unclassified.Christensenellaceae, unclassified.Chroococcidiopsaceae, unclassified.Clostridiaceae, unclassified.Clostridiales, unclassified.Comamonadaceae, unclassified.Coriobacteriales.Incertae.Sedis..EUPATH_0009256_Bacteria_Actinobacteriota_Coriobacteriia_Coriobacteriales_Coriobacteriales_Incertae_Sedis_., unclassified.Coriobacteriales, unclassified.Corynebacteriaceae, unclassified.Desulfovibrionaceae, unclassified.Devosiaceae, unclassified.Diplorickettsiaceae, unclassified.Eggerthellaceae, unclassified.Enterobacterales, unclassified.Enterobacteriaceae, unclassified.Enterococcaceae, unclassified.Erwiniaceae, unclassified.Erysipelatoclostridiaceae, unclassified.Erysipelotrichaceae, unclassified.Erysipelotrichales, unclassified.Ethanoligenenaceae, unclassified.Eubacteriaceae, unclassified.Ezakiella, unclassified.Fastidiosipila, unclassified.Fenollaria, unclassified.Fimbriimonadaceae, unclassified.Finegoldia, unclassified.Firmicutes, unclassified.Flavobacteriaceae, unclassified.Flavobacteriales, unclassified.Fusobacteriaceae, unclassified.Gaiellales, unclassified.Gallicola, unclassified.Gammaproteobacteria, unclassified.Gemmataceae, unclassified.Hungateiclostridiaceae, unclassified.Hydrogenoanaerobacterium, unclassified.Izemoplasmatales, unclassified.JI49D030, unclassified.KD3.93, unclassified.Kapabacteriales, unclassified.Lachnospirales, unclassified.Lactobacillales, unclassified.Leuconostocaceae, unclassified.Lineage.IV, unclassified.MAT.CR.H4.C10, unclassified.Methylococcaceae, unclassified.Micavibrionales, unclassified.Microbacteriaceae, unclassified.Micrococcaceae, unclassified.Micrococcales, unclassified.Moraxellaceae, unclassified.Murdochiella, unclassified.Muribaculaceae, unclassified.NS9.marine.group, unclassified.Neisseriaceae, unclassified.OLB14, unclassified.Opitutales, unclassified.Oscillospirales, unclassified.Oxalobacteraceae, unclassified.P5D1.392, unclassified.PLTA13, unclassified.Paenibacillaceae, unclassified.Paracaedibacteraceae, unclassified.Parachlamydiaceae, unclassified.Parvimonas, unclassified.Pasteurellaceae, unclassified.Peptococcaceae, unclassified.Peptostreptococcaceae, unclassified.Peptostreptococcales.Tissierellales, unclassified.Planctomycetales, unclassified.Planococcaceae, unclassified.Prevotellaceae, unclassified.Promicromonosporaceae, unclassified.Propionibacteriaceae, unclassified.Proteobacteria, unclassified.Pseudomonadaceae, unclassified.Puniceicoccaceae, unclassified.RF39, unclassified.Rhizobiaceae, unclassified.Rhizobiales.Incertae.Sedis..EUPATH_0009256_Bacteria_Proteobacteria_Alphaproteobacteria_Rhizobiales_Rhizobiales_Incertae_Sedis_., unclassified.Rhizobiales, unclassified.Rhodobacteraceae, unclassified.Rhodocyclaceae, unclassified.Rickettsiales, unclassified.Rikenellaceae, unclassified.Ruminiclostridium, unclassified.ST.12K33, unclassified.Saccharimonadales, unclassified.Sandaracinaceae, unclassified.Selenomonadaceae, unclassified.Simkaniaceae, unclassified.Solimonadaceae, unclassified.Sphingobacteriales, unclassified.Sphingomonadaceae, unclassified.Staphylococcaceae, unclassified.Streptococcaceae, unclassified.Succinivibrionaceae, unclassified.Sutterellaceae, unclassified.TRA3.20, unclassified.Tannerellaceae, unclassified.Tepidimicrobium, unclassified.UCG.010, unclassified.UCG.011, unclassified.Veillonellaceae, unclassified.Veillonellales.Selenomonadales, unclassified.Vermiphilaceae, unclassified.Verrucomicrobiales, unclassified.Verrucomicrobiota, unclassified.Victivallaceae, unclassified.W5053, unclassified.WPS.2, unclassified.Weeksellaceae, unclassified.Xanthobacteraceae, unclassified.Yersiniaceae, unclassified..Clostridium..methylpentosum.group, unclassified.vadinBE97</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.427361 INFO::Total filtered features with variance filtering: 0</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.427897 INFO::Filtered feature names from variance filtering:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.428349 INFO::Running selected normalization method: TSS</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.48964 INFO::Applying z-score to standardize continuous metadata</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.492653 INFO::Running selected transform method: LOG</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.566547 INFO::Running selected analysis method: LM</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.567372 INFO::Fitting model to feature number 1, Actinomyces</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.572248 INFO::Fitting model to feature number 2, Agathobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.576733 INFO::Fitting model to feature number 3, Akkermansia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.58122 INFO::Fitting model to feature number 4, Alistipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.585648 INFO::Fitting model to feature number 5, Anaerostipes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.590046 INFO::Fitting model to feature number 6, Atopobium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.594424 INFO::Fitting model to feature number 7, Bacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.599537 INFO::Fitting model to feature number 8, Barnesiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.604165 INFO::Fitting model to feature number 9, Bifidobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.608614 INFO::Fitting model to feature number 10, Bilophila</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.613031 INFO::Fitting model to feature number 11, Blautia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.617481 INFO::Fitting model to feature number 12, Butyricicoccus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.621901 INFO::Fitting model to feature number 13, CAG.352</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.626329 INFO::Fitting model to feature number 14, Campylobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.630732 INFO::Fitting model to feature number 15, Christensenellaceae.R.7.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.635151 INFO::Fitting model to feature number 16, Clostridioides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.639643 INFO::Fitting model to feature number 17, Clostridium.sensu.stricto.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.644057 INFO::Fitting model to feature number 18, Colidextribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.648453 INFO::Fitting model to feature number 19, Collinsella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.652907 INFO::Fitting model to feature number 20, Coprobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.6573 INFO::Fitting model to feature number 21, Coprococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.661693 INFO::Fitting model to feature number 22, DTU089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.666089 INFO::Fitting model to feature number 23, Dialister</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.670487 INFO::Fitting model to feature number 24, Dorea</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.674912 INFO::Fitting model to feature number 25, Eggerthella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.679278 INFO::Fitting model to feature number 26, Eisenbergiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.683712 INFO::Fitting model to feature number 27, Enterococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.688115 INFO::Fitting model to feature number 28, Erysipelatoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.692612 INFO::Fitting model to feature number 29, Erysipelotrichaceae.UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.697077 INFO::Fitting model to feature number 30, Escherichia.Shigella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.701493 INFO::Fitting model to feature number 31, Faecalibacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.70588 INFO::Fitting model to feature number 32, Family.XIII.AD3011.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.710257 INFO::Fitting model to feature number 33, Flavonifractor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.714663 INFO::Fitting model to feature number 34, Fusicatenibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.71911 INFO::Fitting model to feature number 35, Fusobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.723564 INFO::Fitting model to feature number 36, Gemella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.728011 INFO::Fitting model to feature number 37, Granulicatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.732445 INFO::Fitting model to feature number 38, Haemophilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.736866 INFO::Fitting model to feature number 39, Holdemania</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.741243 INFO::Fitting model to feature number 40, Hungatella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.745669 INFO::Fitting model to feature number 41, Incertae.Sedis..EUPATH_0009256_Bacteria_Firmicutes_Clostridia_Oscillospirales_Ruminococcaceae_Incertae_Sedis.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.750061 INFO::Fitting model to feature number 42, Intestinibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.754478 INFO::Fitting model to feature number 43, Intestinimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.758938 INFO::Fitting model to feature number 44, Klebsiella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.763357 INFO::Fitting model to feature number 45, Lachnoclostridium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.767732 INFO::Fitting model to feature number 46, Lachnospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.772128 INFO::Fitting model to feature number 47, Lachnospiraceae.FCS020.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.776505 INFO::Fitting model to feature number 48, Lachnospiraceae.ND3007.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.780883 INFO::Fitting model to feature number 49, Lachnospiraceae.NK4A136.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.78528 INFO::Fitting model to feature number 50, Lachnospiraceae.UCG.001</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.789682 INFO::Fitting model to feature number 51, Lachnospiraceae.UCG.004</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.794076 INFO::Fitting model to feature number 52, Lachnospiraceae.UCG.009</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.798466 INFO::Fitting model to feature number 53, Lachnospiraceae.UCG.010</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.802835 INFO::Fitting model to feature number 54, Lactobacillus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.807283 INFO::Fitting model to feature number 55, Lactococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.811844 INFO::Fitting model to feature number 56, Mannheimia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.816529 INFO::Fitting model to feature number 57, Megasphaera</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.820967 INFO::Fitting model to feature number 58, Monoglobus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.825391 INFO::Fitting model to feature number 59, Neisseria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.829814 INFO::Fitting model to feature number 60, Odoribacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.834168 INFO::Fitting model to feature number 61, Oscillibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.838535 INFO::Fitting model to feature number 62, Oscillospira</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.842893 INFO::Fitting model to feature number 63, Parabacteroides</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.847253 INFO::Fitting model to feature number 64, Parasutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.851658 INFO::Fitting model to feature number 65, Prevotella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.85603 INFO::Fitting model to feature number 66, Pseudomonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.860484 INFO::Fitting model to feature number 67, Romboutsia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.864845 INFO::Fitting model to feature number 68, Roseburia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.869245 INFO::Fitting model to feature number 69, Rothia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.873648 INFO::Fitting model to feature number 70, Ruminococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.878071 INFO::Fitting model to feature number 71, Sellimonas</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.882449 INFO::Fitting model to feature number 72, Solobacterium</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.88683 INFO::Fitting model to feature number 73, Staphylococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.8912 INFO::Fitting model to feature number 74, Streptococcus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.895636 INFO::Fitting model to feature number 75, Subdoligranulum</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.900049 INFO::Fitting model to feature number 76, Sutterella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.90443 INFO::Fitting model to feature number 77, Terrisporobacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.908819 INFO::Fitting model to feature number 78, Turicibacter</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.91319 INFO::Fitting model to feature number 79, Tuzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.917614 INFO::Fitting model to feature number 80, Tyzzerella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.922076 INFO::Fitting model to feature number 81, UBA1819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.926458 INFO::Fitting model to feature number 82, UCG.002</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.930885 INFO::Fitting model to feature number 83, UCG.003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.93531 INFO::Fitting model to feature number 84, UCG.005</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.939664 INFO::Fitting model to feature number 85, Veillonella</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.944092 INFO::Fitting model to feature number 86, X.Clostridium..innocuum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.948529 INFO::Fitting model to feature number 87, X.Eubacterium..eligens.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.95293 INFO::Fitting model to feature number 88, X.Eubacterium..fissicatena.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.957337 INFO::Fitting model to feature number 89, X.Eubacterium..hallii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.961746 INFO::Fitting model to feature number 90, X.Eubacterium..siraeum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.9661 INFO::Fitting model to feature number 91, X.Eubacterium..ventriosum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.970493 INFO::Fitting model to feature number 92, X.Eubacterium..xylanophilum.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.974873 INFO::Fitting model to feature number 93, X.Ruminococcus..gauvreauii.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.979309 INFO::Fitting model to feature number 94, X.Ruminococcus..gnavus.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.983711 INFO::Fitting model to feature number 95, X.Ruminococcus..torques.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.988131 INFO::Fitting model to feature number 96, unclassified.Chloroplast</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.992519 INFO::Fitting model to feature number 97, unclassified.Clostridia.UCG.014</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:46.996932 INFO::Fitting model to feature number 98, unclassified.Clostridia</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.001349 INFO::Fitting model to feature number 99, unclassified.Clostridia.vadinBB60.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.005765 INFO::Fitting model to feature number 100, unclassified.Gastranaerophilales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.010139 INFO::Fitting model to feature number 101, unclassified.Lachnospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.014531 INFO::Fitting model to feature number 102, unclassified.Mitochondria</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.018951 INFO::Fitting model to feature number 103, unclassified.Oscillospiraceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.023343 INFO::Fitting model to feature number 104, unclassified.Peptoniphilus</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.04276 INFO::Fitting model to feature number 105, unclassified.Rhodospirillales</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.047579 INFO::Fitting model to feature number 106, unclassified.Ruminococcaceae</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.052022 INFO::Fitting model to feature number 107, unclassified..Eubacterium..coprostanoligenes.group</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.068367 INFO::Counting total values for each feature</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.073809 INFO::Writing filtered data to file /tmp/RtmpSSLDrC/maaslin17961f373946/features/filtered_data.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.289173 INFO::Writing filtered, normalized data to file /tmp/RtmpSSLDrC/maaslin17961f373946/features/filtered_data_norm.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.510416 INFO::Writing filtered, normalized, transformed data to file /tmp/RtmpSSLDrC/maaslin17961f373946/features/filtered_data_norm_transformed.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.86932 INFO::Writing residuals to file /tmp/RtmpSSLDrC/maaslin17961f373946/fits/residuals.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.940015 INFO::Writing fitted values to file /tmp/RtmpSSLDrC/maaslin17961f373946/fits/fitted.rds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.964216 INFO::Writing all results to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin17961f373946/all_results.tsv</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2024-06-07 02:28:47.966029 INFO::Writing the significant results (those which are less than or equal to the threshold of 0.250000 ) to file (ordered by increasing q-values): /tmp/RtmpSSLDrC/maaslin17961f373946/significant_results.tsv</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

