<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="MicrobiomeDB">
<title>Beta Diversity • MicrobiomeDB</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Beta Diversity">
<meta property="og:description" content="MicrobiomeDB">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.0</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Beta Diversity</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/vignettes/betadiv.Rmd" class="external-link"><code>vignettes/betadiv.Rmd</code></a></small>
      <div class="d-none name"><code>betadiv.Rmd</code></div>
    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/microbiomeDB/MicrobiomeDB" class="external-link">MicrobiomeDB</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning: replacing previous import 'S4Arrays::makeNindexFromArrayViewport' by</span></span>
<span><span class="co">#&gt; 'DelayedArray::makeNindexFromArrayViewport' when loading 'SummarizedExperiment'</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; ── <span style="font-weight: bold;">Attaching core tidyverse packages</span> ──────────────────────── tidyverse 2.0.0 ──</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">dplyr    </span> 1.1.4     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">readr    </span> 2.1.5</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">forcats  </span> 1.0.0     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">stringr  </span> 1.5.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">ggplot2  </span> 3.5.1     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tibble   </span> 3.2.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">lubridate</span> 1.9.3     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tidyr    </span> 1.3.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">purrr    </span> 1.0.2     </span></span>
<span><span class="co">#&gt; ── <span style="font-weight: bold;">Conflicts</span> ────────────────────────────────────────── tidyverse_conflicts() ──</span></span>
<span><span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">filter()</span> masks <span style="color: #0000BB;">stats</span>::filter()</span></span>
<span><span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">lag()</span>    masks <span style="color: #0000BB;">stats</span>::lag()</span></span>
<span><span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Use the conflicted package (<span style="color: #0000BB; font-style: italic;">&lt;http://conflicted.r-lib.org/&gt;</span>) to force all conflicts to become errors</span></span></code></pre></div>
<div class="section level2">
<h2 id="what-is-beta-diversity">What is Beta Diversity?<a class="anchor" aria-label="anchor" href="#what-is-beta-diversity"></a>
</h2>
<p>Beta diversity measures the dissimilarity or diversity between
different microbial communities. In the context of microbiome studies,
it quantifies how microbial compositions vary across samples.
Understanding beta diversity allows researchers to explore the unique
features of each sample and identify patterns in microbial community
structure.</p>
</div>
<div class="section level2">
<h2 id="why-care-about-beta-diversity">Why Care About Beta Diversity?<a class="anchor" aria-label="anchor" href="#why-care-about-beta-diversity"></a>
</h2>
<p>Researchers are interested in beta diversity for several reasons:</p>
<p><strong>Ecological Insights</strong>: Beta diversity helps uncover
the ecological differences between microbial communities in different
environments or conditions.</p>
<p><strong>Disease Studies</strong>: In medical research, beta diversity
can highlight variations in microbial communities associated with health
or disease states.</p>
<p><strong>Community Dynamics</strong>: Studying beta diversity provides
information about how microbial communities change over time or in
response to specific factors.</p>
</div>
<div class="section level2">
<h2 id="how-is-beta-diversity-calculated">How is Beta Diversity Calculated?<a class="anchor" aria-label="anchor" href="#how-is-beta-diversity-calculated"></a>
</h2>
<p>Beta Diversity can be calculated by first producing a dissimilarity
matrix for all samples and then applying a dimensional reduction
technique to the dissimilarity matrix.</p>
<p>This package offers flexibility in calculating beta diversity by
providing multiple dissimilarity matrix options:</p>
<div class="section level4">
<h4 id="bray-curtis-dissimilarity">Bray-Curtis Dissimilarity<a class="anchor" aria-label="anchor" href="#bray-curtis-dissimilarity"></a>
</h4>
<p>The Bray-Curtis algorithm measures compositional dissimilarity based
on both the presence and abundance of taxa. It calculates the normalized
absolute differences in taxon abundance between two samples, providing a
metric that ranges from 0 (complete similarity) to 1 (complete
dissimilarity).</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## first lets find some interesting data</span></span>
<span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="fu"><a href="https://microbiomedb.github.io/microbiomeData/reference/getCuratedDatasetNames.html" class="external-link">getCuratedDatasetNames</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;  [1] "Anopheles_albimanus"     "BONUS"                  </span></span>
<span><span class="co">#&gt;  [3] "Bangladesh"              "DailyBaby"              </span></span>
<span><span class="co">#&gt;  [5] "DiabImmune"              "ECAM"                   </span></span>
<span><span class="co">#&gt;  [7] "EcoCF"                   "FARMM"                  </span></span>
<span><span class="co">#&gt;  [9] "GEMS1"                   "HMP_V1V3"               </span></span>
<span><span class="co">#&gt; [11] "HMP_V3V5"                "HMP_WGS"                </span></span>
<span><span class="co">#&gt; [13] "Leishmaniasis"           "MALED_2yr"              </span></span>
<span><span class="co">#&gt; [15] "MALED_diarrhea"          "MORDOR"                 </span></span>
<span><span class="co">#&gt; [17] "Malaysia_helminth"       "NICU_NEC"               </span></span>
<span><span class="co">#&gt; [19] "PIH_Uganda"              "PretermInfantResistome1"</span></span>
<span><span class="co">#&gt; [21] "PretermInfantResistome2" "UgandaMaternal"</span></span>
<span></span>
<span><span class="fu">getCollectionNames</span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_WGS.html" class="external-link">HMP_WGS</a></span><span class="op">)</span></span>
<span><span class="co">#&gt;  [1] "WGS 4th level EC metagenome abundance data"              </span></span>
<span><span class="co">#&gt;  [2] "WGS Metagenome enzyme pathway abundance data"            </span></span>
<span><span class="co">#&gt;  [3] "WGS Metagenome enzyme pathway coverage data"             </span></span>
<span><span class="co">#&gt;  [4] "WGS Genus"                                               </span></span>
<span><span class="co">#&gt;  [5] "WGS Species"                                             </span></span>
<span><span class="co">#&gt;  [6] "WGS Family"                                              </span></span>
<span><span class="co">#&gt;  [7] "WGS Order"                                               </span></span>
<span><span class="co">#&gt;  [8] "WGS Phylum"                                              </span></span>
<span><span class="co">#&gt;  [9] "WGS Class"                                               </span></span>
<span><span class="co">#&gt; [10] "WGS Normalized number of taxon-specific sequence matches"</span></span>
<span><span class="co">#&gt; [11] "WGS Kingdom"</span></span>
<span></span>
<span><span class="co">## grab a collection we like</span></span>
<span><span class="va">HMP_WGS_species</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_WGS.html" class="external-link">HMP_WGS</a></span>, <span class="st">'WGS Species'</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## get a betaDiv ComputeResult</span></span>
<span><span class="va">betaDiv</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/betaDiv-methods.html">betaDiv</a></span><span class="op">(</span><span class="va">HMP_WGS_species</span>, method <span class="op">=</span> <span class="st">"bray"</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:14.829383 Received df table with 741 samples and 704 taxa.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:15.155728 Computed dissimilarity matrix.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.089393 Finished ordination step.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.11976 Beta diversity computation completed with parameters recordIdColumn= Metagenomic_sequencing_assay_Id , method = bray , k = 2 , verbose = TRUE</span></span></code></pre></div>
</div>
<div class="section level4">
<h4 id="jaccard-dissimilarity">Jaccard Dissimilarity<a class="anchor" aria-label="anchor" href="#jaccard-dissimilarity"></a>
</h4>
<p>Measures dissimilarity based on the presence-absence of taxa. It
quantifies the proportion of taxa that are not shared between two
samples.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">HMP_WGS_species</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_WGS.html" class="external-link">HMP_WGS</a></span>, <span class="st">'WGS Species'</span><span class="op">)</span></span>
<span><span class="va">betaDiv</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/betaDiv-methods.html">betaDiv</a></span><span class="op">(</span><span class="va">HMP_WGS_species</span>, method <span class="op">=</span> <span class="st">"jaccard"</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.269667 Received df table with 741 samples and 704 taxa.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.581029 Computed dissimilarity matrix.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.769338 Finished ordination step.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.780671 Beta diversity computation completed with parameters recordIdColumn= Metagenomic_sequencing_assay_Id , method = jaccard , k = 2 , verbose = TRUE</span></span></code></pre></div>
</div>
<div class="section level4">
<h4 id="jensen-shannon-divergence-jsd">Jensen-Shannon Divergence (JSD)<a class="anchor" aria-label="anchor" href="#jensen-shannon-divergence-jsd"></a>
</h4>
<p>Captures dissimilarity considering both abundance and
presence-absence information. It is a symmetric version of the
Kullback-Leibler Divergence, providing a measure of dissimilarity
between probability distributions.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">HMP_WGS_species</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_WGS.html" class="external-link">HMP_WGS</a></span>, <span class="st">'WGS Species'</span><span class="op">)</span></span>
<span><span class="va">betaDiv</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/betaDiv-methods.html">betaDiv</a></span><span class="op">(</span><span class="va">HMP_WGS_species</span>, method <span class="op">=</span> <span class="st">"jsd"</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:16.914771 Received df table with 741 samples and 704 taxa.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:23.70449 Computed dissimilarity matrix.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:23.903281 Finished ordination step.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-24 17:37:23.90968 Beta diversity computation completed with parameters recordIdColumn= Metagenomic_sequencing_assay_Id , method = jsd , k = 2 , verbose = TRUE</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="principal-coordinate-analysis-pcoa">Principal Coordinate Analysis (PCoA):<a class="anchor" aria-label="anchor" href="#principal-coordinate-analysis-pcoa"></a>
</h3>
<p>PCoA is a dimensional reduction technique applied to the
dissimilarity matrix, providing a visual representation of the
relationships between samples in a lower-dimensional space. It
transforms the dissimilarity matrix into a set of orthogonal axes
(principal coordinates) that capture the maximum variance in the data.
The PCoA plot allows researchers to visualize the spatial arrangement of
samples, aiding in the interpretation of beta diversity. The
<strong>MicrobiomeDB</strong> package performs PCoA as part of the
<code>betaDiv</code> method.</p>
</div>
</div>
<div class="section level2">
<h2 id="interpreting-pcoa-results">Interpreting PCoA Results<a class="anchor" aria-label="anchor" href="#interpreting-pcoa-results"></a>
</h2>
<p>The following code will produce a PCoA plot:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## choose one or more metadata variables to integrate with the compute result</span></span>
<span><span class="va">betaDiv_withMetadata</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getComputeResultWithMetadata.html">getComputeResultWithMetadata</a></span><span class="op">(</span></span>
<span>  <span class="va">betaDiv</span>, </span>
<span>  <span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_WGS.html" class="external-link">HMP_WGS</a></span>, </span>
<span>  metadataVariables <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'host_body_habitat'</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## plot beta diversity</span></span>
<span><span class="fu">ggplot2</span><span class="fu">::</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">betaDiv_withMetadata</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x<span class="op">=</span><span class="va">Axis1</span>, y<span class="op">=</span><span class="va">Axis2</span>, color<span class="op">=</span><span class="va">host_body_habitat</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/labs.html" class="external-link">labs</a></span><span class="op">(</span>y<span class="op">=</span> <span class="st">"Axis 2"</span>, x <span class="op">=</span> <span class="st">"Axis 1"</span>,</span>
<span>        title<span class="op">=</span><span class="st">"Beta diversity by body site"</span>,</span>
<span>        caption<span class="op">=</span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste0</a></span><span class="op">(</span><span class="st">"produced on "</span>, <span class="fu"><a href="https://rdrr.io/r/base/Sys.time.html" class="external-link">Sys.time</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span></span>
<span>  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggtheme.html" class="external-link">theme_bw</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<p><img src="betadiv_files/figure-html/unnamed-chunk-5-1.png" width="700"></p>
<p>The PCoA plot visually represents the dissimilarity between samples.
Each point on the plot corresponds to a sample, and the position of the
points reflects their relationships based on beta diversity. Here’s how
to interpret the PCoA plot:</p>
<div class="section level3">
<h3 id="axes-representation">Axes Representation:<a class="anchor" aria-label="anchor" href="#axes-representation"></a>
</h3>
<p>The axes (principal coordinates) on the PCoA plot represent the
dimensions of maximum variance in the dissimilarity matrix.</p>
<p>The distance between points on the plot reflects the dissimilarity
between corresponding samples.</p>
<p>Each axis explains a certain percentage of the total variation in the
data.</p>
</div>
<div class="section level3">
<h3 id="interpreting-axis-direction">Interpreting Axis Direction:<a class="anchor" aria-label="anchor" href="#interpreting-axis-direction"></a>
</h3>
<p>The direction of the axes indicates the major patterns of
dissimilarity in the data.</p>
<p>Samples that cluster together on the plot are more similar to each
other, while those farther apart are more dissimilar.</p>
</div>
<div class="section level3">
<h3 id="percentage-of-variance">Percentage of Variance:<a class="anchor" aria-label="anchor" href="#percentage-of-variance"></a>
</h3>
<p>Check the percentage of variance explained by each axis. Higher
percentages indicate that the axis captures more information about the
dissimilarity between samples.</p>
</div>
<div class="section level3">
<h3 id="biological-interpretation">Biological Interpretation:<a class="anchor" aria-label="anchor" href="#biological-interpretation"></a>
</h3>
<p>Interpret the biological meaning of the sample clustering. Are there
distinct groups or trends in the data?<br>
Color the PCoA plot with another variable to learn more about these
patterns!</p>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
