<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="MicrobiomeDB">
<title>Correlation • MicrobiomeDB</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Correlation">
<meta property="og:description" content="MicrobiomeDB">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.0.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">



<link href="correlation_files/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="correlation_files/htmlwidgets-1.6.4/htmlwidgets.js"></script><script src="correlation_files/bipartitenetwork-binding-0.0.0.9000/bipartitenetwork.js"></script><script src="correlation_files/d3v5-5.9.2/d3.min.js"></script><div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Correlation</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/vignettes/correlation.Rmd" class="external-link"><code>vignettes/correlation.Rmd</code></a></small>
      <div class="d-none name"><code>correlation.Rmd</code></div>
    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/microbiomeDB/MicrobiomeDB" class="external-link">MicrobiomeDB</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; ── <span style="font-weight: bold;">Attaching core tidyverse packages</span> ──────────────────────── tidyverse 2.0.0 ──</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">dplyr    </span> 1.1.4     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">readr    </span> 2.1.5</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">forcats  </span> 1.0.0     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">stringr  </span> 1.5.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">ggplot2  </span> 3.5.1     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tibble   </span> 3.2.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">lubridate</span> 1.9.3     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tidyr    </span> 1.3.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">purrr    </span> 1.0.2     </span></span>
<span><span class="co">#&gt; ── <span style="font-weight: bold;">Conflicts</span> ────────────────────────────────────────── tidyverse_conflicts() ──</span></span>
<span><span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">filter()</span> masks <span style="color: #0000BB;">stats</span>::filter()</span></span>
<span><span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">lag()</span>    masks <span style="color: #0000BB;">stats</span>::lag()</span></span>
<span><span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Use the conflicted package (<span style="color: #0000BB; font-style: italic;">&lt;http://conflicted.r-lib.org/&gt;</span>) to force all conflicts to become errors</span></span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://r.igraph.org/" class="external-link">igraph</a></span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Attaching package: 'igraph'</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:lubridate':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     %--%, union</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:dplyr':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     as_data_frame, groups, union</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:purrr':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     compose, simplify</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following object is masked from 'package:tidyr':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     crossing</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following object is masked from 'package:tibble':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     as_data_frame</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:stats':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     decompose, spectrum</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following object is masked from 'package:base':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     union</span></span></code></pre></div>
<div class="section level2">
<h2 id="what-is-a-correlation-analysis">What is a Correlation Analysis?<a class="anchor" aria-label="anchor" href="#what-is-a-correlation-analysis"></a>
</h2>
<p>Correlations are useful for identifying relationships between
variables. They are very helpful for identifying biomarkers and
functional associations, among other things.</p>
<p>Biomarkers in microbiome data are microbial taxa or features that
exhibit a significant correlation with particular conditions, traits, or
variables in sample metadata. Identifying these biomarkers allows
researchers to uncover associations between the microbiome and external
factors, such as age, disease status, or environmental conditions.
Biomarkers serve as indicators of specific biological or clinical
phenomena within microbial communities.</p>
<p>Functional associations in microbiome data refer to relationships
between taxonomic relative abundances and abundances of specific
pathways or other functional features. Understanding these associations
allows researchers to explore how the taxonomic composition of microbial
communities influences functional capabilities.</p>
</div>
<div class="section level2">
<h2 id="why-care-about-biomarker-discovery">Why Care About Biomarker Discovery?<a class="anchor" aria-label="anchor" href="#why-care-about-biomarker-discovery"></a>
</h2>
<p>Researchers are interested in biomarker discovery for several
reasons:</p>
<p><strong>Diagnostic Insights</strong>: Biomarkers can serve as
potential diagnostic indicators, helping identify microbial patterns
associated with specific conditions.</p>
<p><strong>Predictive Modeling</strong>: Understanding biomarker
correlations enables predictive modeling of microbial responses to
external factors.</p>
<p><strong>Biological Significance</strong>: Biomarkers provide insights
into the biological significance of microbial community variations in
response to different conditions.</p>
</div>
<div class="section level2">
<h2 id="why-care-about-functional-associations">Why Care About Functional Associations?<a class="anchor" aria-label="anchor" href="#why-care-about-functional-associations"></a>
</h2>
<p><strong>Biological Insights</strong>: Explore how changes in
taxonomic composition may impact the functional potential of microbial
communities.</p>
<p><strong>Pathway-Level Analysis</strong>: Understand how specific
pathways or functional features correlate with taxonomic abundance,
providing pathway-level insights.</p>
<p><strong>Predictive Modeling</strong>: Assessing functional
associations aids in predicting microbial functional responses to
environmental changes or perturbations.</p>
</div>
<div class="section level2">
<h2 id="how-are-correlations-calculated">How are Correlations Calculated?<a class="anchor" aria-label="anchor" href="#how-are-correlations-calculated"></a>
</h2>
<p>This package employs correlation analysis between microbial taxonomic
abundances and sample metadata or abundances of pathways or other
functional data using the following approach:</p>
<p>Users can choose either Spearman or Pearson correlation for the
analysis. Both will produce a correlation coefficient and a p-value
indicating statistical significance.</p>
<div class="section level3">
<h3 id="spearman-correlation">Spearman Correlation<a class="anchor" aria-label="anchor" href="#spearman-correlation"></a>
</h3>
<p>Use when the relationship between variables is monotonic but not
necessarily linear. Suitable for non-linear associations.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## first lets find some interesting data</span></span>
<span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="fu"><a href="https://microbiomedb.github.io/microbiomeData/reference/getCuratedDatasetNames.html" class="external-link">getCuratedDatasetNames</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;  [1] "Anopheles_albimanus"     "BONUS"                  </span></span>
<span><span class="co">#&gt;  [3] "Bangladesh"              "DailyBaby"              </span></span>
<span><span class="co">#&gt;  [5] "DiabImmune"              "ECAM"                   </span></span>
<span><span class="co">#&gt;  [7] "EcoCF"                   "FARMM"                  </span></span>
<span><span class="co">#&gt;  [9] "GEMS1"                   "HMP_MGX"                </span></span>
<span><span class="co">#&gt; [11] "HMP_V1V3"                "HMP_V3V5"               </span></span>
<span><span class="co">#&gt; [13] "Leishmaniasis"           "MALED_2yr"              </span></span>
<span><span class="co">#&gt; [15] "MALED_diarrhea"          "MORDOR"                 </span></span>
<span><span class="co">#&gt; [17] "Malaysia_helminth"       "NICU_NEC"               </span></span>
<span><span class="co">#&gt; [19] "PIH_Uganda"              "PretermInfantResistome1"</span></span>
<span><span class="co">#&gt; [21] "PretermInfantResistome2" "UgandaMaternal"</span></span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="fu">getCollectionNames</span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_MGX.html" class="external-link">HMP_MGX</a></span><span class="op">)</span></span>
<span><span class="co">#&gt;  [1] "Shotgun metagenomics 4th level EC metagenome abundance data"              </span></span>
<span><span class="co">#&gt;  [2] "Shotgun metagenomics Metagenome enzyme pathway abundance data"            </span></span>
<span><span class="co">#&gt;  [3] "Shotgun metagenomics Metagenome enzyme pathway coverage data"             </span></span>
<span><span class="co">#&gt;  [4] "Shotgun metagenomics Genus"                                               </span></span>
<span><span class="co">#&gt;  [5] "Shotgun metagenomics Species"                                             </span></span>
<span><span class="co">#&gt;  [6] "Shotgun metagenomics Family"                                              </span></span>
<span><span class="co">#&gt;  [7] "Shotgun metagenomics Order"                                               </span></span>
<span><span class="co">#&gt;  [8] "Shotgun metagenomics Phylum"                                              </span></span>
<span><span class="co">#&gt;  [9] "Shotgun metagenomics Class"                                               </span></span>
<span><span class="co">#&gt; [10] "Shotgun metagenomics Normalized number of taxon-specific sequence matches"</span></span>
<span><span class="co">#&gt; [11] "Shotgun metagenomics Kingdom"</span></span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## grab a collection of interest</span></span>
<span><span class="va">HMP_MGX_species</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_MGX.html" class="external-link">HMP_MGX</a></span>, <span class="st">"Shotgun metagenomics Species"</span>, continuousMetadataOnly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## get a correlation ComputeResult</span></span>
<span><span class="co">## this is not necessarily to recommend spearman for metadata. </span></span>
<span><span class="co">## it is simply exemplary. Always look at your data!</span></span>
<span><span class="va">species_vs_metadata</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/correlation-methods.html">correlation</a></span><span class="op">(</span><span class="va">HMP_MGX_species</span>, method <span class="op">=</span> <span class="st">'spearman'</span><span class="op">)</span> </span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:01.17511 Removed 7 records with no data.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:01.208922 Removed 7 records with no data.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:01.250555 Removed 7 records with no data.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.585544 Completed correlation with method=spearman. Formatting results.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.586487 Received df table with 734 samples and 316 features with values.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.590663 Correlation computation completed with parameters recordIdColumn= Metagenomic_sequencing_assay_Id , method =  spearman</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="pearson-correlation">Pearson Correlation<a class="anchor" aria-label="anchor" href="#pearson-correlation"></a>
</h3>
<p>Use when the relationship between variables is linear. Suitable for
assessing linear associations.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## grab two collections of interest, in this case species level data and pathway abundance data</span></span>
<span><span class="va">HMP_MGX_species</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_MGX.html" class="external-link">HMP_MGX</a></span>, <span class="st">"Shotgun metagenomics Species"</span><span class="op">)</span></span>
<span><span class="va">HMP_MGX_pathways</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/HMP_MGX.html" class="external-link">HMP_MGX</a></span>, <span class="st">"Shotgun metagenomics Metagenome enzyme pathway abundance data"</span> <span class="op">)</span></span>
<span></span>
<span><span class="co">## get a correlation ComputeResult</span></span>
<span><span class="co">## this is not necessarily to recommend pearson for functional data.</span></span>
<span><span class="co">## it is simply exemplary. Always look at your data!</span></span>
<span><span class="va">pathway_vs_species</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/correlation-methods.html">correlation</a></span><span class="op">(</span><span class="va">HMP_MGX_species</span>, <span class="va">HMP_MGX_pathways</span>, method <span class="op">=</span> <span class="st">'pearson'</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.783298 Removed 7 records with no data.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.810986 Removed 7 records with no data.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.953594 Removed 7 records with no data.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.964409 Received first df table with 734 samples and 319 features with values.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.964655 Received second df table with 741 samples and 371 features with values.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:02.964895 Found 734 samples in common between data1 and data2. Only these samples will be used.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:03.38223 Completed correlation with method=pearson. Formatting results.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-05-23 16:20:03.387836 Correlation computation completed with parameters recordIdColumn= Metagenomic_sequencing_assay_Id , method =  pearson</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="interpreting-results">Interpreting Results<a class="anchor" aria-label="anchor" href="#interpreting-results"></a>
</h2>
<p><strong>Correlation Coefficients</strong>: Assess the strength and
direction of correlations. Positive coefficients indicate positive
correlations, while negative coefficients indicate negative
correlations.</p>
<p><strong>p-values and Adjusted p-values</strong>: Identify biomarkers
with statistically significant correlations, considering adjustments for
multiple testing.</p>
<p>You can extract these metrics and sort and filter results by
them:</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## you can extract network metrics </span></span>
<span><span class="va">pathway_vs_species.metrics</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html" class="external-link">as_tibble</a></span><span class="op">(</span></span>
<span>  <span class="fu"><a href="../reference/getComputeResult.html">getComputeResult</a></span><span class="op">(</span></span>
<span>    <span class="va">pathway_vs_species</span>,</span>
<span>    correlationCoefThreshold <span class="op">=</span> <span class="fl">0.8</span>,</span>
<span>    pValueThreshold <span class="op">=</span> <span class="fl">0.05</span></span>
<span>  <span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## it's also easy to sort and filter these network metrics</span></span>
<span><span class="co">## begin by renaming columns</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">pathway_vs_species.metrics</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'species'</span>, <span class="st">'pathway'</span>, <span class="st">'correlationCoef'</span>, <span class="st">'pValue'</span><span class="op">)</span></span>
<span><span class="va">pathway_vs_species.metrics</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">species</span> <span class="op">==</span> <span class="st">"Faecalibacterium prausnitzii"</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">correlationCoef</span> <span class="op">&gt;</span> <span class="fl">0.5</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>  <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">correlationCoef</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 1 × 4</span></span></span>
<span><span class="co">#&gt;   species                      pathway                    correlationCoef pValue</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                        <span style="color: #949494; font-style: italic;">&lt;fct&gt;</span>                                <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>  <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> Faecalibacterium prausnitzii PWY-5177: glutaryl-CoA de…           0.825      0</span></span></code></pre></div>
<p>You can also visualize them with custom htmlwidgets:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## now plot the network</span></span>
<span><span class="co">## filters can be applied based on correlation coefficient and p-value</span></span>
<span><span class="co">## renders an interactive htmlwidget, using a predetermined layout</span></span>
<span><span class="fu"><a href="../reference/correlationNetwork.html">correlationNetwork</a></span><span class="op">(</span></span>
<span>  <span class="va">pathway_vs_species</span>,</span>
<span>  correlationCoefThreshold <span class="op">=</span> <span class="fl">0.8</span>,</span>
<span>  pValueThreshold <span class="op">=</span> <span class="fl">0.05</span></span>
<span><span class="op">)</span></span></code></pre></div>
<div class="bipartitenetwork html-widget html-fill-item" id="htmlwidget-ac96cb3ee4656e2e9ec3" style="width:700px;height:432.632880098888px;"></div>
<script type="application/json" data-for="htmlwidget-ac96cb3ee4656e2e9ec3">{"x":{"data":{"links":[{"source":"Escherichia coli","target":"ARG+POLYAMINE-SYN: superpathway of arginine and polyamine biosynthesis","value":0.8378910219553921,"pValue":0},{"source":"Escherichia coli","target":"FAO-PWY: fatty acid &beta;-oxidation I","value":0.9986457672087068,"pValue":0},{"source":"Escherichia coli","target":"FUC-RHAMCAT-PWY: superpathway of fucose and rhamnose degradation","value":0.9523425242599398,"pValue":0},{"source":"Escherichia coli","target":"FUCCAT-PWY: fucose degradation","value":0.843447580867717,"pValue":0},{"source":"Escherichia coli","target":"GALACTARDEG-PWY: D-galactarate degradation I","value":0.9969776783312223,"pValue":0},{"source":"Escherichia coli","target":"GLUCARDEG-PWY: D-glucarate degradation I","value":0.9985291706441431,"pValue":0},{"source":"Escherichia coli","target":"GLUCARGALACTSUPER-PWY: superpathway of D-glucarate and D-galactarate degradation","value":0.9969776783312223,"pValue":0},{"source":"Escherichia coli","target":"GLYCOL-GLYOXDEG-PWY: superpathway of glycol metabolism and degradation","value":0.8574128838069022,"pValue":0},{"source":"Escherichia coli","target":"GLYOXYLATE-BYPASS: glyoxylate cycle","value":0.9373168137910555,"pValue":0},{"source":"Escherichia coli","target":"KDO-NAGLIPASYN-PWY: superpathway of (Kdo)2-lipid A biosynthesis","value":0.9002132023263255,"pValue":0},{"source":"Escherichia coli","target":"KETOGLUCONMET-PWY: ketogluconate metabolism","value":0.9981472205626841,"pValue":0},{"source":"Escherichia coli","target":"METHGLYUT-PWY: superpathway of methylglyoxal degradation","value":0.9331731164439296,"pValue":0},{"source":"Escherichia coli","target":"P105-PWY: TCA cycle IV (2-oxoglutarate decarboxylase)","value":0.9286297629489619,"pValue":0},{"source":"Streptococcus parasanguinis","target":"P125-PWY: superpathway of (R,R)-butanediol biosynthesis","value":0.8417282849625227,"pValue":0},{"source":"Fusobacterium periodonticum","target":"P163-PWY: L-lysine fermentation to acetate and butanoate","value":0.8347776325124552,"pValue":0},{"source":"Escherichia coli","target":"P221-PWY: octane oxidation","value":0.9846559491561279,"pValue":0},{"source":"Escherichia coli","target":"P461-PWY: hexitol fermentation to lactate, formate, ethanol and acetate","value":0.876345100646296,"pValue":0},{"source":"Escherichia coli","target":"POLYAMSYN-PWY: superpathway of polyamine biosynthesis I","value":0.8625820090325145,"pValue":0},{"source":"Escherichia coli","target":"PWY-4702: phytate degradation I","value":0.8638734895545219,"pValue":0},{"source":"Monoglobus pectinilyticus","target":"PWY-5101: L-isoleucine biosynthesis II","value":0.8042956002303102,"pValue":0},{"source":"Escherichia coli","target":"PWY-5136: fatty acid &beta;-oxidation II (peroxisome)","value":0.9979936558219484,"pValue":0},{"source":"Escherichia coli","target":"PWY-5138: unsaturated, even numbered fatty acid &beta;-oxidation","value":0.9847273609610459,"pValue":0},{"source":"Faecalibacterium prausnitzii","target":"PWY-5177: glutaryl-CoA degradation","value":0.8247879574443185,"pValue":0},{"source":"Escherichia coli","target":"PWY-561: superpathway of glyoxylate cycle and fatty acid degradation","value":0.9987553577002238,"pValue":0},{"source":"Escherichia coli","target":"PWY-5675: nitrate reduction V (assimilatory)","value":0.8318549888390029,"pValue":0},{"source":"Anaerotruncus colihominis","target":"PWY-5677: succinate fermentation to butanoate","value":0.8022217216346673,"pValue":0},{"source":"Escherichia coli","target":"PWY-5723: Rubisco shunt","value":0.8895629927733554,"pValue":0},{"source":"Escherichia coli","target":"PWY-5845: superpathway of menaquinol-9 biosynthesis","value":0.8476176423961407,"pValue":0},{"source":"Escherichia coli","target":"PWY-5850: superpathway of menaquinol-6 biosynthesis I","value":0.9939899480637638,"pValue":0},{"source":"Escherichia coli","target":"PWY-5860: superpathway of demethylmenaquinol-6 biosynthesis I","value":0.9962726420841463,"pValue":0},{"source":"Escherichia coli","target":"PWY-5862: superpathway of demethylmenaquinol-9 biosynthesis","value":0.8896245347122579,"pValue":0},{"source":"Escherichia coli","target":"PWY-5896: superpathway of menaquinol-10 biosynthesis","value":0.9939899480637638,"pValue":0},{"source":"Flavonifractor plautii","target":"PWY-6318: L-phenylalanine degradation IV (mammalian, via side chain)","value":0.8839532246569202,"pValue":0},{"source":"Escherichia coli","target":"PWY-6353: purine nucleotides degradation II (aerobic)","value":0.8562775625511659,"pValue":0},{"source":"Bacteroides thetaiotaomicron","target":"PWY-6478: GDP-D-glycero-&alpha;-D-manno-heptose biosynthesis","value":0.8590491526118877,"pValue":0},{"source":"Escherichia coli","target":"PWY-6531: mannitol cycle","value":0.9621812370487759,"pValue":0},{"source":"Escherichia coli","target":"PWY-6606: guanosine nucleotides degradation II","value":0.9222033218345683,"pValue":0},{"source":"Escherichia coli","target":"PWY-6731: starch degradation III","value":0.9088487950979812,"pValue":0},{"source":"Campylobacter concisus","target":"PWY-6748: nitrate reduction VII (denitrification)","value":0.8550507738310021,"pValue":0},{"source":"Escherichia coli","target":"PWY-6891: thiazole biosynthesis II (Bacillus)","value":0.8449065802925679,"pValue":0},{"source":"Campylobacter concisus","target":"PWY-7031: protein N-glycosylation (bacterial)","value":0.8869628763874353,"pValue":0},{"source":"Escherichia coli","target":"PWY-7046: 4-coumarate degradation (anaerobic)","value":0.999714035787891,"pValue":0},{"source":"Prevotella melaninogenica","target":"PWY-7200: superpathway of pyrimidine deoxyribonucleoside salvage","value":0.815383059044164,"pValue":0},{"source":"Escherichia coli","target":"PWY-7204: pyridoxal 5'-phosphate salvage II (plants)","value":0.9978691223964024,"pValue":0},{"source":"Escherichia coli","target":"PWY-7209: superpathway of pyrimidine ribonucleosides degradation","value":0.994736969826856,"pValue":0},{"source":"Escherichia coli","target":"PWY-7269: NAD/NADP-NADH/NADPH mitochondrial interconversion (yeast)","value":0.9985944547961759,"pValue":0},{"source":"Bacteroides stercoris","target":"PWY-7312: dTDP-D-&beta;-fucofuranose biosynthesis","value":0.9133558294091612,"pValue":0},{"source":"Escherichia coli","target":"PWY-7315: dTDP-N-acetylthomosamine biosynthesis","value":0.8014295333907402,"pValue":0},{"source":"Corynebacterium matruchotii","target":"PWY-7384: anaerobic energy metabolism (invertebrates, mitochondrial)","value":0.8150822325597555,"pValue":0},{"source":"Corynebacterium matruchotii","target":"PWY-7389: superpathway of anaerobic energy metabolism (invertebrates)","value":0.815064669530158,"pValue":0},{"source":"Cardiobacterium hominis","target":"PWY-7391: isoprene biosynthesis II (engineered)","value":0.8160642327411342,"pValue":0},{"source":"Corynebacterium matruchotii","target":"PWY1G-0: mycothiol biosynthesis","value":0.8761249033298735,"pValue":0},{"source":"Actinomyces sp. oral taxon 448","target":"PWY490-3: nitrate reduction VI (assimilatory)","value":0.9137060691981583,"pValue":0},{"source":"Flavonifractor plautii","target":"PWY66-391: fatty acid &beta;-oxidation VI (peroxisome)","value":0.8589161407387568,"pValue":0},{"source":"Escherichia coli","target":"SALVADEHYPOX-PWY: adenosine nucleotides degradation II","value":0.9576229243929869,"pValue":0},{"source":"Escherichia coli","target":"TCA-GLYOX-BYPASS: superpathway of glyoxylate bypass and TCA","value":0.8910115638617523,"pValue":0}],"column1NodeIds":["Escherichia coli","Streptococcus parasanguinis","Fusobacterium periodonticum","Monoglobus pectinilyticus","Faecalibacterium prausnitzii","Anaerotruncus colihominis","Flavonifractor plautii","Bacteroides thetaiotaomicron","Campylobacter concisus","Prevotella melaninogenica","Bacteroides stercoris","Corynebacterium matruchotii","Cardiobacterium hominis","Actinomyces sp. oral taxon 448"],"column2NodeIds":["ARG+POLYAMINE-SYN: superpathway of arginine and polyamine biosynthesis","FAO-PWY: fatty acid &beta;-oxidation I","FUC-RHAMCAT-PWY: superpathway of fucose and rhamnose degradation","FUCCAT-PWY: fucose degradation","GALACTARDEG-PWY: D-galactarate degradation I","GLUCARDEG-PWY: D-glucarate degradation I","GLUCARGALACTSUPER-PWY: superpathway of D-glucarate and D-galactarate degradation","GLYCOL-GLYOXDEG-PWY: superpathway of glycol metabolism and degradation","GLYOXYLATE-BYPASS: glyoxylate cycle","KDO-NAGLIPASYN-PWY: superpathway of (Kdo)2-lipid A biosynthesis","KETOGLUCONMET-PWY: ketogluconate metabolism","METHGLYUT-PWY: superpathway of methylglyoxal degradation","P105-PWY: TCA cycle IV (2-oxoglutarate decarboxylase)","P125-PWY: superpathway of (R,R)-butanediol biosynthesis","P163-PWY: L-lysine fermentation to acetate and butanoate","P221-PWY: octane oxidation","P461-PWY: hexitol fermentation to lactate, formate, ethanol and acetate","POLYAMSYN-PWY: superpathway of polyamine biosynthesis I","PWY-4702: phytate degradation I","PWY-5101: L-isoleucine biosynthesis II","PWY-5136: fatty acid &beta;-oxidation II (peroxisome)","PWY-5138: unsaturated, even numbered fatty acid &beta;-oxidation","PWY-5177: glutaryl-CoA degradation","PWY-561: superpathway of glyoxylate cycle and fatty acid degradation","PWY-5675: nitrate reduction V (assimilatory)","PWY-5677: succinate fermentation to butanoate","PWY-5723: Rubisco shunt","PWY-5845: superpathway of menaquinol-9 biosynthesis","PWY-5850: superpathway of menaquinol-6 biosynthesis I","PWY-5860: superpathway of demethylmenaquinol-6 biosynthesis I","PWY-5862: superpathway of demethylmenaquinol-9 biosynthesis","PWY-5896: superpathway of menaquinol-10 biosynthesis","PWY-6318: L-phenylalanine degradation IV (mammalian, via side chain)","PWY-6353: purine nucleotides degradation II (aerobic)","PWY-6478: GDP-D-glycero-&alpha;-D-manno-heptose biosynthesis","PWY-6531: mannitol cycle","PWY-6606: guanosine nucleotides degradation II","PWY-6731: starch degradation III","PWY-6748: nitrate reduction VII (denitrification)","PWY-6891: thiazole biosynthesis II (Bacillus)","PWY-7031: protein N-glycosylation (bacterial)","PWY-7046: 4-coumarate degradation (anaerobic)","PWY-7200: superpathway of pyrimidine deoxyribonucleoside salvage","PWY-7204: pyridoxal 5'-phosphate salvage II (plants)","PWY-7209: superpathway of pyrimidine ribonucleosides degradation","PWY-7269: NAD/NADP-NADH/NADPH mitochondrial interconversion (yeast)","PWY-7312: dTDP-D-&beta;-fucofuranose biosynthesis","PWY-7315: dTDP-N-acetylthomosamine biosynthesis","PWY-7384: anaerobic energy metabolism (invertebrates, mitochondrial)","PWY-7389: superpathway of anaerobic energy metabolism (invertebrates)","PWY-7391: isoprene biosynthesis II (engineered)","PWY1G-0: mycothiol biosynthesis","PWY490-3: nitrate reduction VI (assimilatory)","PWY66-391: fatty acid &beta;-oxidation VI (peroxisome)","SALVADEHYPOX-PWY: adenosine nucleotides degradation II","TCA-GLYOX-BYPASS: superpathway of glyoxylate bypass and TCA"]}},"evals":[],"jsHooks":[]}</script><p>Finally, for more advanced analysis of the network, you can extract
it as an igraph object:</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## if you extract the network as an igraph object, you can get more detailed metrics</span></span>
<span><span class="va">pathway_vs_species.igraph</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getComputeResult.html">getComputeResult</a></span><span class="op">(</span><span class="va">pathway_vs_species</span>, format <span class="op">=</span> <span class="st">'igraph'</span><span class="op">)</span></span>
<span><span class="va">degree</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/degree.html" class="external-link">degree</a></span><span class="op">(</span><span class="va">pathway_vs_species.igraph</span><span class="op">)</span></span>
<span><span class="va">edgeBT</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/betweenness.html" class="external-link">edge_betweenness</a></span><span class="op">(</span><span class="va">pathway_vs_species.igraph</span><span class="op">)</span></span>
<span><span class="va">pgRank</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/page_rank.html" class="external-link">page_rank</a></span><span class="op">(</span><span class="va">pathway_vs_species.igraph</span><span class="op">)</span></span>
<span><span class="va">coords</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/layout_with_kk.html" class="external-link">layout_with_kk</a></span><span class="op">(</span><span class="va">pathway_vs_species.igraph</span><span class="op">)</span></span>
<span></span>
<span><span class="co">#plot(pathway_vs_species.igraph, layout = coords)</span></span></code></pre></div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
