<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="MicrobiomeDB">
<title>Self Correlation • MicrobiomeDB</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Self Correlation">
<meta property="og:description" content="MicrobiomeDB">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.0</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">



<link href="self-correlation_files/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="self-correlation_files/htmlwidgets-1.6.4/htmlwidgets.js"></script><script src="self-correlation_files/unipartitenetwork-binding-0.0.0.9000/unipartitenetwork.js"></script><script src="self-correlation_files/d3v5-5.9.2/d3.min.js"></script><div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Self Correlation</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/vignettes/self-correlation.Rmd" class="external-link"><code>vignettes/self-correlation.Rmd</code></a></small>
      <div class="d-none name"><code>self-correlation.Rmd</code></div>
    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/microbiomeDB/MicrobiomeDB" class="external-link">MicrobiomeDB</a></span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning: replacing previous import 'S4Arrays::makeNindexFromArrayViewport' by</span></span>
<span><span class="co">#&gt; 'DelayedArray::makeNindexFromArrayViewport' when loading 'SummarizedExperiment'</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyverse.tidyverse.org" class="external-link">tidyverse</a></span><span class="op">)</span></span>
<span><span class="co">#&gt; ── <span style="font-weight: bold;">Attaching core tidyverse packages</span> ──────────────────────── tidyverse 2.0.0 ──</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">dplyr    </span> 1.1.4     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">readr    </span> 2.1.5</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">forcats  </span> 1.0.0     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">stringr  </span> 1.5.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">ggplot2  </span> 3.5.0     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tibble   </span> 3.2.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">lubridate</span> 1.9.3     <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">tidyr    </span> 1.3.1</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="color: #0000BB;">purrr    </span> 1.0.2     </span></span>
<span><span class="co">#&gt; ── <span style="font-weight: bold;">Conflicts</span> ────────────────────────────────────────── tidyverse_conflicts() ──</span></span>
<span><span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">filter()</span> masks <span style="color: #0000BB;">stats</span>::filter()</span></span>
<span><span class="co">#&gt; <span style="color: #BB0000;">✖</span> <span style="color: #0000BB;">dplyr</span>::<span style="color: #00BB00;">lag()</span>    masks <span style="color: #0000BB;">stats</span>::lag()</span></span>
<span><span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Use the conflicted package (<span style="color: #0000BB; font-style: italic;">&lt;http://conflicted.r-lib.org/&gt;</span>) to force all conflicts to become errors</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://r.igraph.org/" class="external-link">igraph</a></span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Attaching package: 'igraph'</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:lubridate':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     %--%, union</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:dplyr':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     as_data_frame, groups, union</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:purrr':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     compose, simplify</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following object is masked from 'package:tidyr':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     crossing</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following object is masked from 'package:tibble':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     as_data_frame</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:stats':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     decompose, spectrum</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; The following object is masked from 'package:base':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     union</span></span></code></pre></div>
<div class="section level2">
<h2 id="what-is-a-correlation-analysis">What is a Correlation Analysis?<a class="anchor" aria-label="anchor" href="#what-is-a-correlation-analysis"></a>
</h2>
<p>Correlations are useful for identifying relationships between
variables. When you correlate taxonomic abundance variables against
themselves, it essentially produces a microbial network. Producing and
analyzing these networks is complicated by the fact that microbiome data
is inherently compositional. If not properly accounted for, that can
result in a lot of spurious negative correlations.</p>
</div>
<div class="section level2">
<h2 id="why-care-about-microbial-network-analysis">Why Care About Microbial Network Analysis?<a class="anchor" aria-label="anchor" href="#why-care-about-microbial-network-analysis"></a>
</h2>
<p>Scientists delve into microbial network analysis for a large number
of reasons:</p>
<p><strong>Unraveling Complex Interactions</strong>: Microbial network
analysis unveils intricate relationships between microorganisms,
shedding light on their interconnectedness within ecosystems.</p>
<p><strong>Diagnostic Potential</strong>: Identifying key nodes in
microbial networks can unveil potential diagnostic markers, offering a
glimpse into microbial dynamics associated with various diseases or
environmental conditions.</p>
<p><strong>Insights into Functionality</strong>: Microbial network
analysis elucidates functional roles within microbial communities,
offering valuable insights into their ecological and physiological
significance.</p>
<p>From unraveling ecological mysteries to informing medical diagnoses,
microbial network analysis proves indispensable in understanding the
intricate world of microorganisms.</p>
</div>
<div class="section level2">
<h2 id="how-are-correlations-calculated">How are Correlations Calculated?<a class="anchor" aria-label="anchor" href="#how-are-correlations-calculated"></a>
</h2>
<p>This package employs SPARCC for correlation analysis between
microbial taxa. SPARCC calculates correlations that account for the
compositional nature of microbiome data.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## first lets find some interesting data</span></span>
<span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="fu"><a href="https://microbiomedb.github.io/microbiomeData/reference/getCuratedDatasetNames.html" class="external-link">getCuratedDatasetNames</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt;  [1] "Anopheles_albimanus"     "BONUS"                  </span></span>
<span><span class="co">#&gt;  [3] "Bangladesh"              "DailyBaby"              </span></span>
<span><span class="co">#&gt;  [5] "DiabImmune"              "ECAM"                   </span></span>
<span><span class="co">#&gt;  [7] "EcoCF"                   "FARMM"                  </span></span>
<span><span class="co">#&gt;  [9] "GEMS1"                   "HMP_V1V3"               </span></span>
<span><span class="co">#&gt; [11] "HMP_V3V5"                "HMP_WGS"                </span></span>
<span><span class="co">#&gt; [13] "Leishmaniasis"           "MALED_2yr"              </span></span>
<span><span class="co">#&gt; [15] "MALED_diarrhea"          "MORDOR"                 </span></span>
<span><span class="co">#&gt; [17] "Malaysia_helminth"       "NICU_NEC"               </span></span>
<span><span class="co">#&gt; [19] "PIH_Uganda"              "PretermInfantResistome1"</span></span>
<span><span class="co">#&gt; [21] "PretermInfantResistome2" "UgandaMaternal"</span></span>
<span><span class="fu">getCollectionNames</span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/Bangladesh.html" class="external-link">Bangladesh</a></span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] "16S (V4) Genus"   "16S (V4) Class"   "16S (V4) Family"  "16S (V4) Order"  </span></span>
<span><span class="co">#&gt; [5] "16S (V4) Phylum"  "16S (V4) Species" "16S (V4) Kingdom"</span></span>
<span></span>
<span><span class="co">## grab a collection of interest</span></span>
<span><span class="va">Bangladesh_genus</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getCollection.html">getCollection</a></span><span class="op">(</span><span class="fu">microbiomeData</span><span class="fu">::</span><span class="va"><a href="https://microbiomedb.github.io/microbiomeData/reference/Bangladesh.html" class="external-link">Bangladesh</a></span>, <span class="st">"16S (V4) Genus"</span><span class="op">)</span></span>
<span></span>
<span><span class="co">## get a self correlation ComputeResult</span></span>
<span><span class="co">## methods spearman and pearson are options here as well, for data known to not be compositional</span></span>
<span><span class="va">selfCorrelation_genus</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/selfCorrelation-methods.html">selfCorrelation</a></span><span class="op">(</span><span class="va">Bangladesh_genus</span>, method<span class="op">=</span><span class="st">'sparcc'</span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-17 20:14:04.280482 Completed correlation with method=sparcc. Formatting results.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-17 20:14:04.282083 Received df table with 2415 samples and 32 features with abundances.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; 2024-04-17 20:14:04.286702 Correlation computation completed with parameters recordIdColumn= 16S_rRNA_(V4)_assay_Id , method =  sparcc</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="interpreting-results">Interpreting Results<a class="anchor" aria-label="anchor" href="#interpreting-results"></a>
</h2>
<p><strong>Correlation Coefficients</strong>: Assess the strength and
direction of correlations. Positive coefficients indicate positive
correlations, while negative coefficients indicate negative
correlations.</p>
<p><strong>p-values and Adjusted p-values</strong>: Identify biomarkers
with statistically significant correlations, considering adjustments for
multiple testing.</p>
<p>You can extract these metrics and sort and filter results by
them:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## you can extract network metrics </span></span>
<span><span class="va">selfCorrelation_genus.metrics</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html" class="external-link">as_tibble</a></span><span class="op">(</span></span>
<span>    <span class="fu"><a href="../reference/getComputeResult.html">getComputeResult</a></span><span class="op">(</span></span>
<span>        <span class="va">selfCorrelation_genus</span>,</span>
<span>        correlationCoefThreshold <span class="op">=</span> <span class="fl">0.5</span>,</span>
<span>        pValueThreshold <span class="op">=</span> <span class="fl">0.05</span></span>
<span>    <span class="op">)</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## it's also easy to sort and filter these network metrics</span></span>
<span><span class="co">## begin by renaming columns</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/colnames.html" class="external-link">colnames</a></span><span class="op">(</span><span class="va">selfCorrelation_genus.metrics</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'taxa1'</span>, <span class="st">'taxa2'</span>, <span class="st">'correlationCoef'</span>, <span class="st">'pValue'</span><span class="op">)</span></span>
<span><span class="va">selfCorrelation_genus.metrics</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">correlationCoef</span> <span class="op">&gt;</span> <span class="fl">0.5</span><span class="op">)</span> <span class="op"><a href="https://magrittr.tidyverse.org/reference/pipe.html" class="external-link">%&gt;%</a></span></span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/arrange.html" class="external-link">arrange</a></span><span class="op">(</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/desc.html" class="external-link">desc</a></span><span class="op">(</span><span class="va">correlationCoef</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 0 × 4</span></span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># ℹ 4 variables: taxa1 &lt;fct&gt;, taxa2 &lt;fct&gt;, correlationCoef &lt;dbl&gt;, pValue &lt;dbl&gt;</span></span></span></code></pre></div>
<p>You can also visualize them with custom htmlwidgets:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## now plot the network</span></span>
<span><span class="co">## filters can be applied based on correlation coefficient and p-value</span></span>
<span><span class="co">## renders an interactive htmlwidget, using a predetermined layout</span></span>
<span><span class="fu"><a href="../reference/correlationNetwork.html">correlationNetwork</a></span><span class="op">(</span></span>
<span>    <span class="va">selfCorrelation_genus</span>,</span>
<span>    correlationCoefThreshold <span class="op">=</span> <span class="fl">0.5</span>,</span>
<span>    pValueThreshold <span class="op">=</span> <span class="fl">0.05</span></span>
<span><span class="op">)</span></span></code></pre></div>
<div class="unipartitenetwork html-widget html-fill-item" id="htmlwidget-ac96cb3ee4656e2e9ec3" style="width:700px;height:432.632880098888px;"></div>
<script type="application/json" data-for="htmlwidget-ac96cb3ee4656e2e9ec3">{"x":{"data":{"links":[],"nodes":[]}},"evals":[],"jsHooks":[]}</script><p>Finally, for more advanced analysis of the network, you can extract
it as an igraph object:</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## if you extract the network as an igraph object, you can get more detailed metrics</span></span>
<span><span class="va">selfCorrelation_genus.igraph</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getComputeResult.html">getComputeResult</a></span><span class="op">(</span><span class="va">selfCorrelation_genus</span>, format <span class="op">=</span> <span class="st">'igraph'</span><span class="op">)</span></span>
<span><span class="va">degree</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/degree.html" class="external-link">degree</a></span><span class="op">(</span><span class="va">selfCorrelation_genus.igraph</span><span class="op">)</span></span>
<span><span class="va">edgeBT</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/betweenness.html" class="external-link">edge_betweenness</a></span><span class="op">(</span><span class="va">selfCorrelation_genus.igraph</span><span class="op">)</span></span>
<span><span class="va">pgRank</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/page_rank.html" class="external-link">page_rank</a></span><span class="op">(</span><span class="va">selfCorrelation_genus.igraph</span><span class="op">)</span></span>
<span><span class="va">layout</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://r.igraph.org/reference/layout_with_kk.html" class="external-link">layout_with_kk</a></span><span class="op">(</span><span class="va">selfCorrelation_genus.igraph</span><span class="op">)</span></span>
<span></span>
<span><span class="va">layout</span></span>
<span><span class="co">#&gt;      [,1] [,2]</span></span></code></pre></div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.8.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
