<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="MicrobiomeDB">
<title>Importing Data into MicrobiomeDB • MicrobiomeDB</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Importing Data into MicrobiomeDB">
<meta property="og:description" content="MicrobiomeDB">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">MicrobiomeDB</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.1.1</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/alphadiv.html">Alpha Diversity</a>
    <a class="dropdown-item" href="../articles/betadiv.html">Beta Diversity</a>
    <a class="dropdown-item" href="../articles/correlation.html">Correlation</a>
    <a class="dropdown-item" href="../articles/ranked-abundance.html">Ranked Relative Abundances</a>
    <a class="dropdown-item" href="../articles/self-correlation.html">Self Correlation</a>
    <a class="dropdown-item" href="../articles/user-data.html">Importing Data into MicrobiomeDB</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/microbiomeDB/MicrobiomeDB/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Importing Data into MicrobiomeDB</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/microbiomeDB/MicrobiomeDB/blob/HEAD/vignettes/user-data.Rmd" class="external-link"><code>vignettes/user-data.Rmd</code></a></small>
      <div class="d-none name"><code>user-data.Rmd</code></div>
    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/microbiomeDB/MicrobiomeDB" class="external-link">MicrobiomeDB</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="introduction">Importing Data<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<p>The MicrobiomeDB package supports importing data from the following
formats: * <a href="#treesummarizedexperiment">TreeSummarizedExperiment</a> * <a href="#phyloseq">phyloseq</a> * <a href="#biom">BIOM</a> * <a href="#qiime2">QIIME2 output files</a> * <a href="#mothur">Mothur output
files</a> * <a href="#dada2">dada2 output files</a> * <a href="#humann">HUMAnN output files</a></p>
<p>For formats other than <code>TreeSummarizedExperiment</code> the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. You can install it like:</p>
<pre><code><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu">install</span><span class="op">(</span><span class="st">"mia"</span><span class="op">)</span></span></code></pre>
<p>Please note that Linux and MacOS with source-level installations
require the ‘gsl’ system dependency for <code>mia</code>.</p>
<p>On Debian or Ubuntu</p>
<pre><code>sudo apt-get install -y libgsl-dev</code></pre>
<p>On Fedora, CentOS or RHEL</p>
<pre><code>sudo yum install libgsl-devel</code></pre>
<p>On macOS (source installations are not common on macOS, so this step
is not usually necessary)</p>
<pre><code>brew install gsl</code></pre>
<div class="section level3">
<h3 id="treesummarizedexperiment">TreeSummarizedExperiment<a class="anchor" aria-label="anchor" href="#treesummarizedexperiment"></a>
</h3>
<p>The MicrobiomeDB package natively supports importing
<code>TreeSummarizedExperiment</code> objects. There is a function
<code>importTreeSummarizedExperiment</code> (as well as the alias
<code>importTreeSE</code>) which will produce an
<code>MbioDataset</code> object from a
<code>TreeSummarizedExperiment</code>. It is used in the following
way:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">GlobalPatterns</span>, package<span class="op">=</span><span class="st">"mia"</span><span class="op">)</span></span>
<span><span class="va">tse</span> <span class="op">&lt;-</span> <span class="va">GlobalPatterns</span></span>
<span></span>
<span><span class="co">## no normalization, with raw values</span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importTreeSummarizedExperiment.html">importTreeSummarizedExperiment</a></span><span class="op">(</span></span>
<span>      <span class="va">tse</span>, </span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## TSS normalization, drop raw values</span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importTreeSummarizedExperiment.html">importTreeSummarizedExperiment</a></span><span class="op">(</span></span>
<span>      <span class="va">tse</span>, </span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"TSS"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">FALSE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## TSS normalization, keep raw values</span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importTreeSummarizedExperiment.html">importTreeSummarizedExperiment</a></span><span class="op">(</span></span>
<span>      <span class="va">tse</span>, </span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"TSS"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="phyloseq">Phyloseq<a class="anchor" aria-label="anchor" href="#phyloseq"></a>
</h3>
<p>First, make sure <a href="https://bioconductor.org/packages/phyloseq" class="external-link">phyloseq</a> is
installed:</p>
<pre><code><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu">install</span><span class="op">(</span><span class="st">"phyloseq"</span><span class="op">)</span></span></code></pre>
<p>Then, in order to import phyloseq objects, the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. Once that is installed (see <a href="#introduction">above</a>), you can do the following:</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">GlobalPatterns</span>, package<span class="op">=</span><span class="st">"phyloseq"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importPhyloseq.html">importPhyloseq</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">GlobalPatterns</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="biom">BIOM<a class="anchor" aria-label="anchor" href="#biom"></a>
</h3>
<p>First, make sure <a href="https://bioconductor.org/packages/biomformat" class="external-link">biomformat</a> is
installed:</p>
<pre><code><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu">install</span><span class="op">(</span><span class="st">"biomformat"</span><span class="op">)</span></span></code></pre>
<p>Then, in order to import biom files and objects, the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. Once that is installed (see <a href="#introduction">above</a>), you can do the following:</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## passing the file name directly</span></span>
<span><span class="va">rich_dense_file</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"rich_dense_otu_table.biom"</span>,</span>
<span>                              package <span class="op">=</span> <span class="st">"biomformat"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importBIOM.html">importBIOM</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">rich_dense_file</span></span>
<span><span class="op">)</span></span>
<span></span>
<span><span class="co">## passing a file already read into R as a biom object</span></span>
<span><span class="va">rich_dense_biom</span> <span class="op">=</span> <span class="fu">biomformat</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/biomformat/man/read_biom.html" class="external-link">read_biom</a></span><span class="op">(</span><span class="va">rich_dense_file</span><span class="op">)</span></span>
<span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importBIOM.html">importBIOM</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">rich_dense_biom</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="qiime2">QIIME2<a class="anchor" aria-label="anchor" href="#qiime2"></a>
</h3>
<p>In order to import QIIME2 output files, the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. Once that is installed (see <a href="#introduction">above</a>), you can do the following:</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">featureTableFile</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"table.qza"</span>, package <span class="op">=</span> <span class="st">"mia"</span><span class="op">)</span></span>
<span><span class="va">taxonomyTableFile</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"taxonomy.qza"</span>, package <span class="op">=</span> <span class="st">"mia"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importQIIME2.html">importQIIME2</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">featureTableFile</span>, </span>
<span>      <span class="va">taxonomyTableFile</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="mothur">Mothur<a class="anchor" aria-label="anchor" href="#mothur"></a>
</h3>
<p>In order to import Mother output files, the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. Once that is installed (see <a href="#introduction">above</a>), you can do the following:</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">counts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"mothur_example.shared"</span>, package <span class="op">=</span> <span class="st">"mia"</span><span class="op">)</span></span>
<span><span class="va">taxa</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"mothur_example.cons.taxonomy"</span>, package <span class="op">=</span> <span class="st">"mia"</span><span class="op">)</span></span>
<span><span class="va">meta</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"mothur_example.design"</span>, package <span class="op">=</span> <span class="st">"mia"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importMothur.html">importMothur</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">counts</span>, </span>
<span>      <span class="va">taxa</span>, </span>
<span>      <span class="va">meta</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="dada2">dada2<a class="anchor" aria-label="anchor" href="#dada2"></a>
</h3>
<p>First, make sure <a href="https://bioconductor.org/packages/dada2" class="external-link">dada2</a> is
installed:</p>
<pre><code><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu">install</span><span class="op">(</span><span class="st">"dada2"</span><span class="op">)</span></span></code></pre>
<p>Then, in order to import dada2 output files, the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. Once that is installed (see <a href="#introduction">above</a>), you can do the following:</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fnF</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"sam1F.fastq.gz"</span>, package<span class="op">=</span><span class="st">"dada2"</span><span class="op">)</span></span>
<span><span class="va">fnR</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"sam1R.fastq.gz"</span>, package<span class="op">=</span><span class="st">"dada2"</span><span class="op">)</span></span>
<span><span class="va">dadaF</span> <span class="op">&lt;-</span> <span class="fu">dada2</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/dada2/man/dada.html" class="external-link">dada</a></span><span class="op">(</span><span class="va">fnF</span>, selfConsist<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; Initializing error rates to maximum possible estimate.</span></span>
<span><span class="co">#&gt; selfConsist step 1 .</span></span>
<span><span class="co">#&gt;    selfConsist step 2</span></span>
<span><span class="co">#&gt;    selfConsist step 3</span></span>
<span><span class="co">#&gt;    selfConsist step 4</span></span>
<span><span class="co">#&gt; Convergence after  4  rounds.</span></span></code></pre></div>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dadaR</span> <span class="op">&lt;-</span> <span class="fu">dada2</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/dada2/man/dada.html" class="external-link">dada</a></span><span class="op">(</span><span class="va">fnR</span>, selfConsist<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; Initializing error rates to maximum possible estimate.</span></span>
<span><span class="co">#&gt; selfConsist step 1 .</span></span>
<span><span class="co">#&gt;    selfConsist step 2</span></span>
<span><span class="co">#&gt;    selfConsist step 3</span></span>
<span><span class="co">#&gt;    selfConsist step 4</span></span>
<span><span class="co">#&gt; Convergence after  4  rounds.</span></span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importDADA2.html">importDADA2</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">dadaF</span>, </span>
<span>      <span class="va">fnF</span>, </span>
<span>      <span class="va">dadaR</span>, </span>
<span>      <span class="va">fnR</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="humann">HUMAnN<a class="anchor" aria-label="anchor" href="#humann"></a>
</h3>
<p>In order to import HUMAnN output files, the <a href="https://bioconductor.org/packages/mia" class="external-link">mia</a> package is also
required. Once that is installed (see <a href="#introduction">above</a>), you can do the following:</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## this is the merged output file</span></span>
<span><span class="va">file_path</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"humann_output.tsv"</span>, package <span class="op">=</span> <span class="st">"mia"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">mbioDataset</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/importHUMAnN.html">importHUMAnN</a></span><span class="op">(</span></span>
<span>      normalizationMethod <span class="op">=</span> <span class="st">"none"</span>, </span>
<span>      keepRawValues <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      verbose <span class="op">=</span> <span class="cn">TRUE</span>, </span>
<span>      <span class="va">file_path</span></span>
<span><span class="op">)</span></span></code></pre></div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Danielle Callan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
